XRootD clients intermittently disconnected on login, TLS error, errno=104

Stephan Lammel noted intermittent TLS errors reported by XRootD clients, affecting ETF availability testing. He, Bockjoo and I have been puzzling over it, but have not found the underlying issue.

The problem also was discussed in [1736](https://github.com/xrootd/xrootd/issues/1736#issuecomment-1234451819), but it doesn't seem to be solved.

### Description
- Intermittently, the client is disconnected by the redirector with a TLS error immediately after login. This causes a 120 second (by default) delay until the client retries.
- `xrdfs $HOST query config version` will demonstrate the problem
- Client logs show `TLS error rc=-1 ec=5 (error_syscall) errno=104.` and `Socket error while handshaking: [ERROR] TLS error`
- Server logs show only `disc`
- Having IPv6 enabled seems to mask the issue, as the client will immediately try the other protocol

### Reproducibility
- Reliably appears on an EL7 local redirector with TLS enabled. Redirector was running xrootd-5.5.3-1.2.osg36.el7.x86_64.
- Happens roughly 0.5-3% of the time
- Appears between systems on a local network, so not expecting packet loss or similar network issues
- Tried client versions v5.4.2 and v5.5.3 and behavior was the same

Looking at a packet capture of an error, the client and server complete a TLS v1.2 handshake. Then the client sends an kXR_login. The server immediately closes the TCP connection with a RST/ACK. Compared to a successful login, the packet capture to that point looks nearly identical, with only the pid changing.

Enabling heap debugging with mimalloc did not detect any errors. I've not yet tried valgrind, because that would be challenging to run on our production redirector. I'm at a loss for how to debug this further, and would appreciate ideas.

Best regards,
John

<details>
  <summary>Server logs (for the first attempt)</summary>

```
230308 14:31:00.412570 9861 Xrd_Inet: Accepted connection on port 1094 from 306@t3.unl.edu
230308 14:31:00.419385 9861 Xrd_ProtLoad: matched port 1094 protocol xroot
230308 14:31:00.419410 9861 anon.0:306@t3 Xrd_Poll: FD 306 attached to poller 1; num=122
230308 14:31:00.419486 9861 anon.0:306@t3 TLS_Accept: Accepting a TLS connection...
230308 14:31:00.429852 9861 XrdLinkXeq: anon.0:306@t3 connection upgraded to TLSv1.2
230308 14:31:00.430337 9861 XrootdXeq: anon.0:306@t3 disc 0:00:00
230308 14:31:00.430363 9861 anon.0:306@t3 Xrd_Poll: Poller 1 removing FD 306
230308 14:31:00.430375 9861 anon.0:306@t3 Xrd_Poll: FD 306 detached from poller 1; num=121
```
</details>

<details>
  <summary>Client logs</summary>

```
[2023-03-08 14:31:00.398502 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.4.2
[2023-03-08 14:31:00.399069 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
[2023-03-08 14:31:00.399226 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
[2023-03-08 14:31:00.399280 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
[2023-03-08 14:31:00.399296 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
[2023-03-08 14:31:00.399307 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
[2023-03-08 14:31:00.399417 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
[2023-03-08 14:31:00.399445 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.401150 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] URL: root://xrootd-local.unl.edu:1094/
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.401198 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] URL: root://xrootd-local.unl.edu:1094/
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.401230 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] URL: root://xrootd-local.unl.edu:1094/
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.401260 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.401311 -0600][Debug  ][App               ] Executing: query config version 
[2023-03-08 14:31:00.401324 -0600][Dump   ][App               ] Param #00: 'query'
[2023-03-08 14:31:00.401335 -0600][Dump   ][App               ] Param #01: 'config'
[2023-03-08 14:31:00.401344 -0600][Dump   ][App               ] Param #02: 'version'
[2023-03-08 14:31:00.401388 -0600][Dump   ][FileSystem        ] [0x1f89e50@xrootd-local.unl.edu:1094] Sending kXR_query (code: kXR_Qconfig, arg length: 8)
[2023-03-08 14:31:00.401411 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
[2023-03-08 14:31:00.401428 -0600][Debug  ][Poller            ] Available pollers: built-in
[2023-03-08 14:31:00.401447 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
[2023-03-08 14:31:00.401460 -0600][Debug  ][Poller            ] Creating poller: built-in
[2023-03-08 14:31:00.401476 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
[2023-03-08 14:31:00.410943 -0600][Debug  ][Poller            ] Using 1 poller threads
[2023-03-08 14:31:00.410983 -0600][Debug  ][TaskMgr           ] Starting the task manager...
[2023-03-08 14:31:00.411048 -0600][Debug  ][TaskMgr           ] Task manager started
[2023-03-08 14:31:00.411067 -0600][Debug  ][JobMgr            ] Starting the job manager...
[2023-03-08 14:31:00.411217 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
[2023-03-08 14:31:00.411240 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-08 14:31:00 -0600]
[2023-03-08 14:31:00.411253 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_query (code: kXR_Qconfig, arg length: 8)
[2023-03-08 14:31:00.411293 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x1f92480 (message: kXR_query (code: kXR_Qconfig, arg length: 8) ).
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.411351 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] URL: root://xrootd-local.unl.edu:1094
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] User Name: 
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] Password:  
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-08 14:31:00.411390 -0600][Dump   ][Utility           ] Path:      
[2023-03-08 14:31:00.411415 -0600][Debug  ][PostMaster        ] Creating new channel to: root://xrootd-local.unl.edu:1094
[2023-03-08 14:31:00.411449 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
[2023-03-08 14:31:00.411478 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" to be run at: [2023-03-08 14:31:15 -0600]
[2023-03-08 14:31:00.411498 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_query (code: kXR_Qconfig, arg length: 8) (0x1f8d4c0) through substream 0 expecting answer at 0
[2023-03-08 14:31:00.411983 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-08 14:31:00.412050 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-08 14:31:00.412107 -0600][Debug  ][Poller            ] Adding socket 0x1f928b0 to the poller
[2023-03-08 14:31:00.418996 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-08 14:31:00.419048 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-08 14:31:00.419105 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x58000950), 44 bytes
[2023-03-08 14:31:00.419317 -0600][Dump   ][XRootDTransport   ] [msg: 0x58000ab0] Expecting 8 bytes of message body
[2023-03-08 14:31:00.419333 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:31:00.419357 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-08 14:31:00.419374 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-08 14:31:00.419398 -0600][Dump   ][XRootDTransport   ] [msg: 0x58000ab0] Expecting 8 bytes of message body
[2023-03-08 14:31:00.419409 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:31:00.419422 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-08 14:31:00.419438 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-08 14:31:00.419931 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: ?xrd.cc=us&xrd.tz=-6&xrd.appname=xrdfs&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.4.2, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-08 14:31:00.419955 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:31:00.419967 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:31:00.419994 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:31:15 -0600]
[2023-03-08 14:31:00.425877 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:31:00.429813 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:31:00.429968 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-08 14:31:00.430031 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x58000be0), 110 bytes
[2023-03-08 14:31:00.430378 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
[2023-03-08 14:31:00.430403 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
[2023-03-08 14:31:00.430417 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-08 14:31:00.430432 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:17946><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-08 14:31:00.430517 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
[2023-03-08 14:31:00.430533 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
[2023-03-08 14:31:00.430546 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-08 14:33:00 -0600]
[2023-03-08 14:31:15.420987 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:31:15.421038 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:31:30 -0600]
[2023-03-08 14:31:15.421051 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:31:15.421065 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:31:30 -0600]
[2023-03-08 14:31:30.421996 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:31:30.422036 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:31:45 -0600]
[2023-03-08 14:31:30.422048 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:31:30.422062 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:31:45 -0600]
[2023-03-08 14:31:45.424318 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:31:45.424377 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:32:00 -0600]
[2023-03-08 14:31:45.424390 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:31:45.424404 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:32:00 -0600]
[2023-03-08 14:32:00.425379 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:32:00.425419 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:32:15 -0600]
[2023-03-08 14:32:00.425431 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:32:00.425446 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:32:15 -0600]
[2023-03-08 14:32:15.426948 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:32:15.427016 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:32:30 -0600]
[2023-03-08 14:32:15.427035 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:32:15.427053 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:32:30 -0600]
[2023-03-08 14:32:30.429204 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:32:30.429272 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:32:45 -0600]
[2023-03-08 14:32:30.429290 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:32:30.429300 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:32:45 -0600]
[2023-03-08 14:32:45.431482 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:32:45.431561 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:33:00 -0600]
[2023-03-08 14:32:45.431582 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:32:45.431598 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:33:00 -0600]
[2023-03-08 14:33:00.433974 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-08 14:33:00.434472 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-08 14:33:00.434525 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-08 14:33:00.434571 -0600][Debug  ][Poller            ] Adding socket 0x1f928b0 to the poller
[2023-03-08 14:33:00.434604 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-08 14:33:00.434614 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:33:00.434627 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: root://xrootd-local.unl.edu:1094" at [2023-03-08 14:33:15 -0600]
[2023-03-08 14:33:00.434635 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-08 14:33:00.434645 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-08 14:33:15 -0600]
[2023-03-08 14:33:00.435004 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-08 14:33:00.435052 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-08 14:33:00.435106 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x58042950), 44 bytes
[2023-03-08 14:33:00.435414 -0600][Dump   ][XRootDTransport   ] [msg: 0x58041170] Expecting 8 bytes of message body
[2023-03-08 14:33:00.435444 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:33:00.435468 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-08 14:33:00.435479 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-08 14:33:00.435514 -0600][Dump   ][XRootDTransport   ] [msg: 0x5803dce0] Expecting 8 bytes of message body
[2023-03-08 14:33:00.435524 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:33:00.435535 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-08 14:33:00.435547 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-08 14:33:00.436589 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: ?xrd.cc=us&xrd.tz=-6&xrd.appname=xrdfs&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.4.2, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-08 14:33:00.436609 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:33:00.440214 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:33:00.442756 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-08 14:33:00.442864 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-08 14:33:00.442907 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x580008c0), 110 bytes
[2023-03-08 14:33:00.443289 -0600][Dump   ][XRootDTransport   ] [msg: 0x58043690] Expecting 75 bytes of message body
[2023-03-08 14:33:00.443310 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:33:00.443379 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
[2023-03-08 14:33:00.443399 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: da28000048260000a5000000562d0000
[2023-03-08 14:33:00.443408 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
[2023-03-08 14:33:00.443418 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
[2023-03-08 14:33:00.444765 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
[2023-03-08 14:33:01.330423 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x58061520), 136 bytes
[2023-03-08 14:33:01.338711 -0600][Dump   ][XRootDTransport   ] [msg: 0x5803d430] Expecting 3252 bytes of message body
[2023-03-08 14:33:01.338741 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:33:01.338807 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
[2023-03-08 14:33:01.338829 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
[2023-03-08 14:33:01.349005 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0x58064440), 9932 bytes
[2023-03-08 14:33:01.398007 -0600][Dump   ][XRootDTransport   ] [msg: 0x58043690] Expecting 0 bytes of message body
[2023-03-08 14:33:01.398037 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-08 14:33:01.398055 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
[2023-03-08 14:33:01.398129 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
[2023-03-08 14:33:01.398149 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected.
[2023-03-08 14:33:01.398163 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
[2023-03-08 14:33:01.398193 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_query (code: kXR_Qconfig, arg length: 8) (0x1f8d4c0), 32 bytes
[2023-03-08 14:33:01.398219 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_query (code: kXR_Qconfig, arg length: 8) (0x1f8d4c0).
[2023-03-08 14:33:01.398231 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_query (code: kXR_Qconfig, arg length: 8) has been successfully sent.
[2023-03-08 14:33:01.398240 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x1f92480 (message: kXR_query (code: kXR_Qconfig, arg length: 8) ) from out-queu to in-queue.
[2023-03-08 14:33:01.398252 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-08 14:33:01.398620 -0600][Dump   ][XRootDTransport   ] [msg: 0x580008d0] Expecting 7 bytes of message body
[2023-03-08 14:33:01.398651 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0x580008d0 size: 8
[2023-03-08 14:33:01.398708 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0x580008d0 of 15 bytes
[2023-03-08 14:33:01.398731 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0x580008d0.
[2023-03-08 14:33:01.398832 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got a kXR_ok response to request kXR_query (code: kXR_Qconfig, arg length: 8)
[2023-03-08 14:33:01.398867 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Calling MsgHandler: 0x1f92480 (message: kXR_query (code: kXR_Qconfig, arg length: 8) ) with status: [SUCCESS] .
[2023-03-08 14:33:01.398891 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Parsing the response to kXR_query (code: kXR_Qconfig, arg length: 8) as BinaryData
[2023-03-08 14:33:01.398925 -0600][Dump   ][FileSystem        ] [0x1f89e50@xrootd-local.unl.edu:1094] Assigning xrootd-local.unl.edu:1094 as last URL
[2023-03-08 14:33:01.398948 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Destroying MsgHandler: 0x1f92480.
v5.5.3

[2023-03-08 14:33:01.400842 -0600][Debug  ][JobMgr            ] Stopping the job manager...
[2023-03-08 14:33:01.400867 -0600][Dump   ][JobMgr            ] Stopping worker #0...
[2023-03-08 14:33:01.401091 -0600][Dump   ][JobMgr            ] Worker #0 stopped
[2023-03-08 14:33:01.401123 -0600][Dump   ][JobMgr            ] Stopping worker #1...
[2023-03-08 14:33:01.401330 -0600][Dump   ][JobMgr            ] Worker #1 stopped
[2023-03-08 14:33:01.401362 -0600][Dump   ][JobMgr            ] Stopping worker #2...
[2023-03-08 14:33:01.401464 -0600][Dump   ][JobMgr            ] Worker #2 stopped
[2023-03-08 14:33:01.401496 -0600][Debug  ][JobMgr            ] Job manager stopped
[2023-03-08 14:33:01.401519 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
[2023-03-08 14:33:01.401784 -0600][Debug  ][TaskMgr           ] Task manager stopped
[2023-03-08 14:33:01.401817 -0600][Debug  ][Poller            ] Stopping the poller...
[2023-03-08 14:33:01.401963 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: root://xrootd-local.unl.edu:1094"
[2023-03-08 14:33:01.401994 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-08 14:33:01.402021 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:7838><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-08 14:33:01.402159 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
[2023-03-08 14:33:01.402183 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
```
</details>

Good question! I'm able to reproduce the same behavior with `xrdcp roots://$HOST//store/hello_world.txt -`

<details>
  <summary>Client logs</summary>

```
[2023-03-10 15:43:36.899973 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.5.3
[2023-03-10 15:43:36.900575 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
[2023-03-10 15:43:36.900750 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
[2023-03-10 15:43:36.900805 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
[2023-03-10 15:43:36.900823 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
[2023-03-10 15:43:36.900836 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
[2023-03-10 15:43:36.900890 -0600][Dump   ][PlugInMgr         ] Processing: /etc/xrootd/client.plugins.d/recorder.conf
[2023-03-10 15:43:36.900963 -0600][Dump   ][PlugInMgr         ] Settings from '/etc/xrootd/client.plugins.d/recorder.conf': url='*', lib='/usr/lib64/libXrdClRecorder-5.so', enable='false'
[2023-03-10 15:43:36.900979 -0600][Debug  ][PlugInMgr         ] Trying to disable plug-in for '*'
[2023-03-10 15:43:36.901262 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
[2023-03-10 15:43:36.901291 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
[2023-03-10 15:43:36.904563 -0600][Dump   ][App               ] Chunk size: 8388608, parallel chunks 4, streams: 1
[2023-03-10 15:43:36.904587 -0600][Dump   ][App               ] Processing source entry: roots://xrootd-local.unl.edu//store/hello_world.txt, type other, target file: -
[2023-03-10 15:43:36.904744 -0600][Dump   ][Utility           ] Adding job with properties: 'checkSumMode' = 'none', 'checkSumPreset' = '', 'checkSumType' = '', 'chunkSize' = '8388608', 'coerce' = '0', 'continue' = '0', 'cpTimeout' = '0', 'delegate' = '0', 'doServer' = '0', 'dynamicSource' = '0', 'force' = '0', 'initTimeout' = '600', 'makeDir' = '0', 'parallelChunks' = '4', 'posc' = '0', 'preserveXAttr' = '0', 'rmOnBadCksum' = '0', 'source' = 'roots://xrootd-local.unl.edu//store/hello_world.txt', 'target' = '-', 'targetIsDir' = '0', 'thirdParty' = 'none', 'tpcTimeout' = '1800', 'xcp' = '0', 'xcpBlockSize' = '134217728', 'xrate' = '0', 'xrateThreshold' = '0', 'zipAppend' = '0', 'zipArchive' = '0'
[2023-03-10 15:43:36.904780 -0600][Debug  ][Utility           ] CopyProcess: 2 jobs to prepare
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] URL: -
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Protocol:  stdio
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Host Name: 
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Port:      0
[2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Path:      -
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] URL: -
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Protocol:  stdio
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Host Name: 
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Port:      0
[2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Path:      -
[2023-03-10 15:43:36.905049 -0600][Debug  ][Utility           ] Creating a classic copy job, from roots://xrootd-local.unl.edu:1094//store/hello_world.txt to stdio://-
[2023-03-10 15:43:36.905081 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
[2023-03-10 15:43:36.905163 -0600][Debug  ][Utility           ] Opening roots://xrootd-local.unl.edu:1094//store/hello_world.txt for reading
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:43:36.905610 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending an open command
[2023-03-10 15:43:36.905658 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
[2023-03-10 15:43:36.905677 -0600][Debug  ][Poller            ] Available pollers: built-in
[2023-03-10 15:43:36.905694 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
[2023-03-10 15:43:36.905706 -0600][Debug  ][Poller            ] Creating poller: built-in
[2023-03-10 15:43:36.905721 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
[2023-03-10 15:43:36.915954 -0600][Debug  ][Poller            ] Using 1 poller threads
[2023-03-10 15:43:36.915980 -0600][Debug  ][TaskMgr           ] Starting the task manager...
[2023-03-10 15:43:36.916044 -0600][Debug  ][TaskMgr           ] Task manager started
[2023-03-10 15:43:36.916068 -0600][Debug  ][JobMgr            ] Starting the job manager...
[2023-03-10 15:43:36.916220 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
[2023-03-10 15:43:36.916244 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-10 15:43:36 -0600]
[2023-03-10 15:43:36.916257 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 15:43:36.916293 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094/
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:43:36.916419 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://xrootd-local.unl.edu:1094/
[2023-03-10 15:43:36.916458 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
[2023-03-10 15:43:36.916485 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" to be run at: [2023-03-10 15:43:51 -0600]
[2023-03-10 15:43:36.916506 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0) through substream 0 expecting answer at 0
[2023-03-10 15:43:36.916882 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-10 15:43:36.916935 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-10 15:43:36.916996 -0600][Debug  ][Poller            ] Adding socket 0x205ebb0 to the poller
[2023-03-10 15:43:36.924003 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-10 15:43:36.924056 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 15:43:36.924108 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4000950), 44 bytes
[2023-03-10 15:43:36.924344 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000ab0] Expecting 8 bytes of message body
[2023-03-10 15:43:36.924360 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:43:36.924376 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 15:43:36.924393 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-10 15:43:36.924415 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000ab0] Expecting 8 bytes of message body
[2023-03-10 15:43:36.924433 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:43:36.924447 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 15:43:36.924464 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-10 15:43:36.924982 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 15:43:36.925006 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:43:36.925982 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:43:36.926013 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:43:51 -0600]
[2023-03-10 15:43:36.930822 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:43:36.934956 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:43:36.935099 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 15:43:36.935160 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4000be0), 109 bytes
[2023-03-10 15:43:36.935415 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
[2023-03-10 15:43:36.935442 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
[2023-03-10 15:43:36.935455 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-10 15:43:36.935469 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:14642><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-10 15:43:36.935552 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
[2023-03-10 15:43:36.935568 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
[2023-03-10 15:43:36.935580 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-10 15:45:36 -0600]
[2023-03-10 15:43:51.927003 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:43:51.927057 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:06 -0600]
[2023-03-10 15:43:51.927069 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:43:51.927084 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:06 -0600]
[2023-03-10 15:44:06.928441 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:44:06.928488 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:21 -0600]
[2023-03-10 15:44:06.928500 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:44:06.928515 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:21 -0600]
[2023-03-10 15:44:21.929523 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:44:21.929574 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:36 -0600]
[2023-03-10 15:44:21.929588 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:44:21.929603 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:36 -0600]
[2023-03-10 15:44:36.930591 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:44:36.930638 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:51 -0600]
[2023-03-10 15:44:36.930650 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:44:36.930665 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:51 -0600]
[2023-03-10 15:44:51.931649 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:44:51.931704 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:06 -0600]
[2023-03-10 15:44:51.931718 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:44:51.931734 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:06 -0600]
[2023-03-10 15:45:06.932726 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:45:06.932772 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:21 -0600]
[2023-03-10 15:45:06.932784 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:45:06.932799 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:21 -0600]
[2023-03-10 15:45:21.933794 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:45:21.933845 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:36 -0600]
[2023-03-10 15:45:21.933857 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:45:21.933871 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:36 -0600]
[2023-03-10 15:45:36.934863 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-10 15:45:36.935251 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-10 15:45:36.935292 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-10 15:45:36.935344 -0600][Debug  ][Poller            ] Adding socket 0x205ebb0 to the poller
[2023-03-10 15:45:36.935386 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-10 15:45:36.935401 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:45:36.935418 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:51 -0600]
[2023-03-10 15:45:36.935430 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 15:45:36.935445 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:51 -0600]
[2023-03-10 15:45:36.935554 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-10 15:45:36.935587 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 15:45:36.935632 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40430f0), 44 bytes
[2023-03-10 15:45:36.935873 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4020400] Expecting 8 bytes of message body
[2023-03-10 15:45:36.935892 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:36.935906 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 15:45:36.935938 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-10 15:45:36.935961 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4020400] Expecting 8 bytes of message body
[2023-03-10 15:45:36.935973 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:36.935988 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 15:45:36.936003 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-10 15:45:36.936445 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 15:45:36.936467 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:36.938352 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:36.940851 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:36.940970 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 15:45:36.941024 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40401d0), 109 bytes
[2023-03-10 15:45:36.941203 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40430f0] Expecting 75 bytes of message body
[2023-03-10 15:45:36.941221 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:36.941260 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
[2023-03-10 15:45:36.941282 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: c3eb220023490000080700007e552900
[2023-03-10 15:45:36.941293 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
[2023-03-10 15:45:36.941304 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
[2023-03-10 15:45:36.942942 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
[2023-03-10 15:45:37.755591 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40584f0), 136 bytes
[2023-03-10 15:45:37.763666 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40439d0] Expecting 3252 bytes of message body
[2023-03-10 15:45:37.763684 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.763752 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
[2023-03-10 15:45:37.763766 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
[2023-03-10 15:45:37.795794 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4057df0), 9932 bytes
[2023-03-10 15:45:37.846827 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40401d0] Expecting 0 bytes of message body
[2023-03-10 15:45:37.846848 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.846859 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
[2023-03-10 15:45:37.846925 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
[2023-03-10 15:45:37.846951 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected (IPv4).
[2023-03-10 15:45:37.846992 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0), 46 bytes
[2023-03-10 15:45:37.847031 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0).
[2023-03-10 15:45:37.847049 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
[2023-03-10 15:45:37.847060 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
[2023-03-10 15:45:37.847074 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 15:45:37.847561 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4043630] Expecting 21 bytes of message body
[2023-03-10 15:45:37.847579 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0xb4043630 size: 8
[2023-03-10 15:45:37.847592 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4043630] Assigned MsgHandler: 0x205e680.
[2023-03-10 15:45:37.847603 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
[2023-03-10 15:45:37.847644 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0xb4043630 of 29 bytes
[2023-03-10 15:45:37.847663 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0xb4043630.
[2023-03-10 15:45:37.847738 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got kXR_redirect response to message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ): red-xfer8.unl.edu, port 1094
[2023-03-10 15:45:37.847766 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Current server has been assigned as a load-balancer for message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] URL: red-xfer8.unl.edu:1094/
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] URL: fake://fake:111//fake?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316&
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Protocol:  fake
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Host Name: fake
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Port:      111
[2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Path:      /fake
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] URL: fake://fake:111//store/hello_world.txt
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Protocol:  fake
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Host Name: fake
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Port:      111
[2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094/
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:45:37.848039 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Retry at server MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] URL: red-xfer8.unl.edu:1094
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094/
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 15:45:37.848124 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://red-xfer8.unl.edu:1094/
[2023-03-10 15:45:37.848158 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
[2023-03-10 15:45:37.848178 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://red-xfer8.unl.edu:1094/" to be run at: [2023-03-10 15:45:52 -0600]
[2023-03-10 15:45:37.848194 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0) through substream 0 expecting answer at 0
[2023-03-10 15:45:37.848531 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.138]:1094
[2023-03-10 15:45:37.848570 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.138]:1094
[2023-03-10 15:45:37.848618 -0600][Debug  ][Poller            ] Adding socket 0xb0001e00 to the poller
[2023-03-10 15:45:37.848719 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Async connection call returned
[2023-03-10 15:45:37.848746 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 15:45:37.848778 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4000be0), 44 bytes
[2023-03-10 15:45:37.848866 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4039090] Expecting 8 bytes of message body
[2023-03-10 15:45:37.848888 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.848902 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 15:45:37.848922 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Got the server hand shake response (type: server [], protocol version 511)
[2023-03-10 15:45:37.848942 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4039090] Expecting 14 bytes of message body
[2023-03-10 15:45:37.848953 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.848966 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 22 bytes
[2023-03-10 15:45:37.848982 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] kXR_protocol successful (type: server [], protocol version 511)
[2023-03-10 15:45:37.849429 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 15:45:37.849450 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:37.851225 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:37.853330 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 15:45:37.853440 -0600][Info   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 15:45:37.853493 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb40430f0), 109 bytes
[2023-03-10 15:45:37.853639 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000be0] Expecting 75 bytes of message body
[2023-03-10 15:45:37.853653 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.853692 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 83 bytes
[2023-03-10 15:45:37.853712 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Logged in, session: 30335e00410500008700000091e15f00
[2023-03-10 15:45:37.853722 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
[2023-03-10 15:45:37.853733 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending authentication data
[2023-03-10 15:45:37.853762 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Trying to authenticate using gsi
[2023-03-10 15:45:37.879376 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4069560), 136 bytes
[2023-03-10 15:45:37.891120 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40430f0] Expecting 3272 bytes of message body
[2023-03-10 15:45:37.891139 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.891218 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 3280 bytes
[2023-03-10 15:45:37.891234 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending more authentication data for gsi
[2023-03-10 15:45:37.923098 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4068310), 9932 bytes
[2023-03-10 15:45:37.937567 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40698b0] Expecting 0 bytes of message body
[2023-03-10 15:45:37.937586 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 15:45:37.937597 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 8 bytes
[2023-03-10 15:45:37.937981 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] XrdSecProtect loaded.
[2023-03-10 15:45:37.938002 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Authenticated with gsi.
[2023-03-10 15:45:37.938029 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Stream 0 connected (IPv4).
[2023-03-10 15:45:37.938069 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0), 46 bytes
[2023-03-10 15:45:37.938100 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0).
[2023-03-10 15:45:37.938115 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
[2023-03-10 15:45:37.938127 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
[2023-03-10 15:45:37.938139 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 15:45:37.938407 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40788c0] Expecting 87 bytes of message body
[2023-03-10 15:45:37.938424 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb40788c0 size: 8
[2023-03-10 15:45:37.938436 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb40788c0] Assigned MsgHandler: 0x205e680.
[2023-03-10 15:45:37.938447 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
[2023-03-10 15:45:37.938490 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb40788c0 of 95 bytes
[2023-03-10 15:45:37.938506 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb40788c0.
[2023-03-10 15:45:37.938581 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 15:45:37.938613 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) with status: [SUCCESS] .
[2023-03-10 15:45:37.938626 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing the response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) as OpenInfo
[2023-03-10 15:45:37.938637 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing StatInfo in response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 15:45:37.938688 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Open has returned with status [SUCCESS] 
[2023-03-10 15:45:37.938708 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] successfully opened at red-xfer8.unl.edu:1094, handle: 0x0, session id: 1
[2023-03-10 15:45:37.938750 -0600][Debug  ][XRootD            ] Redirect trace-back:
[2023-03-10 15:45:37.938750 -0600][Debug  ][XRootD            ] 	0. Redirected from: roots://xrootd-local.unl.edu:1094//store/hello_world.txt to: roots://red-xfer8.unl.edu:1094/
[2023-03-10 15:45:37.938765 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 15:45:37.938838 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending a read command for handle 0x0 to red-xfer8.unl.edu:1094
[2023-03-10 15:45:37.938873 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 15:45:37.938892 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ).
[2023-03-10 15:45:37.938910 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0) through substream 0 expecting answer at 0
[2023-03-10 15:45:37.938962 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0), 32 bytes
[2023-03-10 15:45:37.938994 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0).
[2023-03-10 15:45:37.939007 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_read (handle: 0x00000000, offset: 0, size: 13) has been successfully sent.
[2023-03-10 15:45:37.939018 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) from out-queu to in-queue.
[2023-03-10 15:45:37.939029 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 15:45:37.939104 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4068400] Expecting 13 bytes of message body
[2023-03-10 15:45:37.939119 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb4068400 size: 8
[2023-03-10 15:45:37.939129 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4068400] Assigned MsgHandler: 0x205e680.
[2023-03-10 15:45:37.939139 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
[2023-03-10 15:45:37.939149 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Will use the raw handler to read body of message 0xb4068400
[2023-03-10 15:45:37.939181 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb4068400 of 21 bytes
[2023-03-10 15:45:37.939196 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb4068400.
[2023-03-10 15:45:37.939256 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 15:45:37.939281 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) with status: [SUCCESS] .
[2023-03-10 15:45:37.939296 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing the response to kXR_read (handle: 0x00000000, offset: 0, size: 13) as ChunkInfo
[2023-03-10 15:45:37.939311 -0600][Dump   ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Got state response for message kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 15:45:37.939337 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
hello world!
[2023-03-10 15:45:37.939371 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending a close command for handle 0x0 to red-xfer8.unl.edu:1094
[2023-03-10 15:45:37.939404 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Sending message kXR_close (handle: 0x00000000)
[2023-03-10 15:45:37.939421 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_close (handle: 0x00000000) ).
[2023-03-10 15:45:37.939439 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_close (handle: 0x00000000) (0x2058a70) through substream 0 expecting answer at 0
[2023-03-10 15:45:37.939479 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_close (handle: 0x00000000) (0x2058a70), 24 bytes
[2023-03-10 15:45:37.939509 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_close (handle: 0x00000000) (0x2058a70).
[2023-03-10 15:45:37.939523 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_close (handle: 0x00000000) has been successfully sent.
[2023-03-10 15:45:37.939533 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_close (handle: 0x00000000) ) from out-queu to in-queue.
[2023-03-10 15:45:37.939544 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 15:45:37.939700 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4063300] Expecting 0 bytes of message body
[2023-03-10 15:45:37.939716 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb4063300 size: 8
[2023-03-10 15:45:37.939726 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4063300] Assigned MsgHandler: 0x205e680.
[2023-03-10 15:45:37.939736 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
[2023-03-10 15:45:37.939747 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb4063300 of 8 bytes
[2023-03-10 15:45:37.939758 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb4063300.
[2023-03-10 15:45:37.939791 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_close (handle: 0x00000000)
[2023-03-10 15:45:37.939808 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_close (handle: 0x00000000) ) with status: [SUCCESS] .
[2023-03-10 15:45:37.939824 -0600][Debug  ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Close returned from red-xfer8.unl.edu:1094 with: [SUCCESS] 
[2023-03-10 15:45:37.939835 -0600][Dump   ][File              ] [0x2059930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Items in the fly 0, queued for recovery 0
[2023-03-10 15:45:37.939855 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
[2023-03-10 15:45:37.940452 -0600][Debug  ][JobMgr            ] Stopping the job manager...
[2023-03-10 15:45:37.940475 -0600][Dump   ][JobMgr            ] Stopping worker #0...
[2023-03-10 15:45:37.940602 -0600][Dump   ][JobMgr            ] Worker #0 stopped
[2023-03-10 15:45:37.940620 -0600][Dump   ][JobMgr            ] Stopping worker #1...
[2023-03-10 15:45:37.940679 -0600][Dump   ][JobMgr            ] Worker #1 stopped
[2023-03-10 15:45:37.940695 -0600][Dump   ][JobMgr            ] Stopping worker #2...
[2023-03-10 15:45:37.940752 -0600][Dump   ][JobMgr            ] Worker #2 stopped
[2023-03-10 15:45:37.940768 -0600][Debug  ][JobMgr            ] Job manager stopped
[2023-03-10 15:45:37.940779 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
[2023-03-10 15:45:37.940882 -0600][Debug  ][TaskMgr           ] Task manager stopped
[2023-03-10 15:45:37.940900 -0600][Debug  ][Poller            ] Stopping the poller...
[2023-03-10 15:45:37.941133 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://red-xfer8.unl.edu:1094/"
[2023-03-10 15:45:37.941159 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Closing the socket
[2023-03-10 15:45:37.941205 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:13868><--><[::ffff:129.93.239.138]:1094> Removing socket from the poller
[2023-03-10 15:45:37.941281 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Destroying stream
[2023-03-10 15:45:37.941300 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Closing the socket
[2023-03-10 15:45:37.941319 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 15:45:37.941331 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-10 15:45:37.941341 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:33348><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-10 15:45:37.941457 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
[2023-03-10 15:45:37.941474 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
```
</details>

Hi JT,

Well, in the trace it looks like roots copy recovered and completed 
successfully. In each cse (roots and https) please trun on tls tracing on 
the server and when a failure happens please post the server loh s we can 
see what it really is complaining about.


On Fri, 10 Mar 2023, jthiltges wrote:

> Good question! I'm able to reproduce the same behavior with `xrdcp roots://$HOST//store/hello_world.txt -`
>
> <details>
>  <summary>Client logs</summary>
>
> ```
> [2023-03-10 15:43:36.899973 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.5.3
> [2023-03-10 15:43:36.900575 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
> [2023-03-10 15:43:36.900750 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
> [2023-03-10 15:43:36.900805 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
> [2023-03-10 15:43:36.900823 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
> [2023-03-10 15:43:36.900836 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
> [2023-03-10 15:43:36.900890 -0600][Dump   ][PlugInMgr         ] Processing: /etc/xrootd/client.plugins.d/recorder.conf
> [2023-03-10 15:43:36.900963 -0600][Dump   ][PlugInMgr         ] Settings from '/etc/xrootd/client.plugins.d/recorder.conf': url='*', lib='/usr/lib64/libXrdClRecorder-5.so', enable='false'
> [2023-03-10 15:43:36.900979 -0600][Debug  ][PlugInMgr         ] Trying to disable plug-in for '*'
> [2023-03-10 15:43:36.901262 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
> [2023-03-10 15:43:36.901291 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
> [2023-03-10 15:43:36.904563 -0600][Dump   ][App               ] Chunk size: 8388608, parallel chunks 4, streams: 1
> [2023-03-10 15:43:36.904587 -0600][Dump   ][App               ] Processing source entry: roots://xrootd-local.unl.edu//store/hello_world.txt, type other, target file: -
> [2023-03-10 15:43:36.904744 -0600][Dump   ][Utility           ] Adding job with properties: 'checkSumMode' = 'none', 'checkSumPreset' = '', 'checkSumType' = '', 'chunkSize' = '8388608', 'coerce' = '0', 'continue' = '0', 'cpTimeout' = '0', 'delegate' = '0', 'doServer' = '0', 'dynamicSource' = '0', 'force' = '0', 'initTimeout' = '600', 'makeDir' = '0', 'parallelChunks' = '4', 'posc' = '0', 'preserveXAttr' = '0', 'rmOnBadCksum' = '0', 'source' = 'roots://xrootd-local.unl.edu//store/hello_world.txt', 'target' = '-', 'targetIsDir' = '0', 'thirdParty' = 'none', 'tpcTimeout' = '1800', 'xcp' = '0', 'xcpBlockSize' = '134217728', 'xrate' = '0', 'xrateThreshold' = '0', 'zipAppend' = '0', 'zipArchive' = '0'
> [2023-03-10 15:43:36.904780 -0600][Debug  ][Utility           ] CopyProcess: 2 jobs to prepare
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.904816 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 15:43:36.904848 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.904888 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 15:43:36.905028 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 15:43:36.905049 -0600][Debug  ][Utility           ] Creating a classic copy job, from roots://xrootd-local.unl.edu:1094//store/hello_world.txt to stdio://-
> [2023-03-10 15:43:36.905081 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
> [2023-03-10 15:43:36.905163 -0600][Debug  ][Utility           ] Opening roots://xrootd-local.unl.edu:1094//store/hello_world.txt for reading
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.905455 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.905491 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.905523 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:43:36.905610 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending an open command
> [2023-03-10 15:43:36.905658 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
> [2023-03-10 15:43:36.905677 -0600][Debug  ][Poller            ] Available pollers: built-in
> [2023-03-10 15:43:36.905694 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
> [2023-03-10 15:43:36.905706 -0600][Debug  ][Poller            ] Creating poller: built-in
> [2023-03-10 15:43:36.905721 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
> [2023-03-10 15:43:36.915954 -0600][Debug  ][Poller            ] Using 1 poller threads
> [2023-03-10 15:43:36.915980 -0600][Debug  ][TaskMgr           ] Starting the task manager...
> [2023-03-10 15:43:36.916044 -0600][Debug  ][TaskMgr           ] Task manager started
> [2023-03-10 15:43:36.916068 -0600][Debug  ][JobMgr            ] Starting the job manager...
> [2023-03-10 15:43:36.916220 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
> [2023-03-10 15:43:36.916244 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-10 15:43:36 -0600]
> [2023-03-10 15:43:36.916257 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 15:43:36.916293 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.916347 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:43:36.916384 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:43:36.916419 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 15:43:36.916458 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 15:43:36.916485 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" to be run at: [2023-03-10 15:43:51 -0600]
> [2023-03-10 15:43:36.916506 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0) through substream 0 expecting answer at 0
> [2023-03-10 15:43:36.916882 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 15:43:36.916935 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 15:43:36.916996 -0600][Debug  ][Poller            ] Adding socket 0x205ebb0 to the poller
> [2023-03-10 15:43:36.924003 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 15:43:36.924056 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 15:43:36.924108 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4000950), 44 bytes
> [2023-03-10 15:43:36.924344 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000ab0] Expecting 8 bytes of message body
> [2023-03-10 15:43:36.924360 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:43:36.924376 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 15:43:36.924393 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 15:43:36.924415 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000ab0] Expecting 8 bytes of message body
> [2023-03-10 15:43:36.924433 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:43:36.924447 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 15:43:36.924464 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 15:43:36.924982 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 15:43:36.925006 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:43:36.925982 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:43:36.926013 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:43:51 -0600]
> [2023-03-10 15:43:36.930822 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:43:36.934956 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:43:36.935099 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 15:43:36.935160 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4000be0), 109 bytes
> [2023-03-10 15:43:36.935415 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
> [2023-03-10 15:43:36.935442 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
> [2023-03-10 15:43:36.935455 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 15:43:36.935469 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:14642><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 15:43:36.935552 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
> [2023-03-10 15:43:36.935568 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
> [2023-03-10 15:43:36.935580 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-10 15:45:36 -0600]
> [2023-03-10 15:43:51.927003 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:43:51.927057 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:06 -0600]
> [2023-03-10 15:43:51.927069 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:43:51.927084 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:06 -0600]
> [2023-03-10 15:44:06.928441 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:44:06.928488 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:21 -0600]
> [2023-03-10 15:44:06.928500 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:44:06.928515 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:21 -0600]
> [2023-03-10 15:44:21.929523 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:44:21.929574 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:36 -0600]
> [2023-03-10 15:44:21.929588 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:44:21.929603 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:36 -0600]
> [2023-03-10 15:44:36.930591 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:44:36.930638 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:44:51 -0600]
> [2023-03-10 15:44:36.930650 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:44:36.930665 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:44:51 -0600]
> [2023-03-10 15:44:51.931649 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:44:51.931704 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:06 -0600]
> [2023-03-10 15:44:51.931718 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:44:51.931734 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:06 -0600]
> [2023-03-10 15:45:06.932726 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:45:06.932772 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:21 -0600]
> [2023-03-10 15:45:06.932784 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:45:06.932799 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:21 -0600]
> [2023-03-10 15:45:21.933794 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:45:21.933845 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:36 -0600]
> [2023-03-10 15:45:21.933857 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:45:21.933871 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:36 -0600]
> [2023-03-10 15:45:36.934863 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 15:45:36.935251 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 15:45:36.935292 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 15:45:36.935344 -0600][Debug  ][Poller            ] Adding socket 0x205ebb0 to the poller
> [2023-03-10 15:45:36.935386 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 15:45:36.935401 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:45:36.935418 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 15:45:51 -0600]
> [2023-03-10 15:45:36.935430 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 15:45:36.935445 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 15:45:51 -0600]
> [2023-03-10 15:45:36.935554 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 15:45:36.935587 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 15:45:36.935632 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40430f0), 44 bytes
> [2023-03-10 15:45:36.935873 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4020400] Expecting 8 bytes of message body
> [2023-03-10 15:45:36.935892 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:36.935906 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 15:45:36.935938 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 15:45:36.935961 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4020400] Expecting 8 bytes of message body
> [2023-03-10 15:45:36.935973 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:36.935988 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 15:45:36.936003 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 15:45:36.936445 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 15:45:36.936467 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:36.938352 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:36.940851 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:36.940970 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 15:45:36.941024 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40401d0), 109 bytes
> [2023-03-10 15:45:36.941203 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40430f0] Expecting 75 bytes of message body
> [2023-03-10 15:45:36.941221 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:36.941260 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 15:45:36.941282 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: c3eb220023490000080700007e552900
> [2023-03-10 15:45:36.941293 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 15:45:36.941304 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
> [2023-03-10 15:45:36.942942 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 15:45:37.755591 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb40584f0), 136 bytes
> [2023-03-10 15:45:37.763666 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40439d0] Expecting 3252 bytes of message body
> [2023-03-10 15:45:37.763684 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.763752 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
> [2023-03-10 15:45:37.763766 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 15:45:37.795794 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xb4057df0), 9932 bytes
> [2023-03-10 15:45:37.846827 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40401d0] Expecting 0 bytes of message body
> [2023-03-10 15:45:37.846848 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.846859 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 15:45:37.846925 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 15:45:37.846951 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 15:45:37.846992 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0), 46 bytes
> [2023-03-10 15:45:37.847031 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0).
> [2023-03-10 15:45:37.847049 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 15:45:37.847060 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 15:45:37.847074 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 15:45:37.847561 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4043630] Expecting 21 bytes of message body
> [2023-03-10 15:45:37.847579 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0xb4043630 size: 8
> [2023-03-10 15:45:37.847592 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4043630] Assigned MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.847603 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
> [2023-03-10 15:45:37.847644 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0xb4043630 of 29 bytes
> [2023-03-10 15:45:37.847663 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0xb4043630.
> [2023-03-10 15:45:37.847738 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got kXR_redirect response to message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ): red-xfer8.unl.edu, port 1094
> [2023-03-10 15:45:37.847766 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Current server has been assigned as a load-balancer for message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] URL: red-xfer8.unl.edu:1094/
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.847798 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] URL: fake://fake:111//fake?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316&
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 15:45:37.847840 -0600][Dump   ][Utility           ] Path:      /fake
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] URL: fake://fake:111//store/hello_world.txt
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 15:45:37.847893 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.847960 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094/
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.847998 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:45:37.848039 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Retry at server MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] URL: red-xfer8.unl.edu:1094
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.848067 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094/
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.848100 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 15:45:37.848124 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://red-xfer8.unl.edu:1094/
> [2023-03-10 15:45:37.848158 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 15:45:37.848178 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://red-xfer8.unl.edu:1094/" to be run at: [2023-03-10 15:45:52 -0600]
> [2023-03-10 15:45:37.848194 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0) through substream 0 expecting answer at 0
> [2023-03-10 15:45:37.848531 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.138]:1094
> [2023-03-10 15:45:37.848570 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.138]:1094
> [2023-03-10 15:45:37.848618 -0600][Debug  ][Poller            ] Adding socket 0xb0001e00 to the poller
> [2023-03-10 15:45:37.848719 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Async connection call returned
> [2023-03-10 15:45:37.848746 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 15:45:37.848778 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4000be0), 44 bytes
> [2023-03-10 15:45:37.848866 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4039090] Expecting 8 bytes of message body
> [2023-03-10 15:45:37.848888 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.848902 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 15:45:37.848922 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Got the server hand shake response (type: server [], protocol version 511)
> [2023-03-10 15:45:37.848942 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4039090] Expecting 14 bytes of message body
> [2023-03-10 15:45:37.848953 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.848966 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 22 bytes
> [2023-03-10 15:45:37.848982 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] kXR_protocol successful (type: server [], protocol version 511)
> [2023-03-10 15:45:37.849429 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 15:45:37.849450 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:37.851225 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:37.853330 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 15:45:37.853440 -0600][Info   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 15:45:37.853493 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb40430f0), 109 bytes
> [2023-03-10 15:45:37.853639 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4000be0] Expecting 75 bytes of message body
> [2023-03-10 15:45:37.853653 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.853692 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 15:45:37.853712 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Logged in, session: 30335e00410500008700000091e15f00
> [2023-03-10 15:45:37.853722 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 15:45:37.853733 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending authentication data
> [2023-03-10 15:45:37.853762 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 15:45:37.879376 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4069560), 136 bytes
> [2023-03-10 15:45:37.891120 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40430f0] Expecting 3272 bytes of message body
> [2023-03-10 15:45:37.891139 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.891218 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 3280 bytes
> [2023-03-10 15:45:37.891234 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 15:45:37.923098 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message:  (0xb4068310), 9932 bytes
> [2023-03-10 15:45:37.937567 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40698b0] Expecting 0 bytes of message body
> [2023-03-10 15:45:37.937586 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 15:45:37.937597 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 15:45:37.937981 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] XrdSecProtect loaded.
> [2023-03-10 15:45:37.938002 -0600][Debug  ][XRootDTransport   ] [red-xfer8.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 15:45:37.938029 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 15:45:37.938069 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0), 46 bytes
> [2023-03-10 15:45:37.938100 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2058bb0).
> [2023-03-10 15:45:37.938115 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 15:45:37.938127 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 15:45:37.938139 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 15:45:37.938407 -0600][Dump   ][XRootDTransport   ] [msg: 0xb40788c0] Expecting 87 bytes of message body
> [2023-03-10 15:45:37.938424 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb40788c0 size: 8
> [2023-03-10 15:45:37.938436 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb40788c0] Assigned MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.938447 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
> [2023-03-10 15:45:37.938490 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb40788c0 of 95 bytes
> [2023-03-10 15:45:37.938506 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb40788c0.
> [2023-03-10 15:45:37.938581 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 15:45:37.938613 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) with status: [SUCCESS] .
> [2023-03-10 15:45:37.938626 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing the response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) as OpenInfo
> [2023-03-10 15:45:37.938637 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing StatInfo in response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 15:45:37.938688 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Open has returned with status [SUCCESS]
> [2023-03-10 15:45:37.938708 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] successfully opened at red-xfer8.unl.edu:1094, handle: 0x0, session id: 1
> [2023-03-10 15:45:37.938750 -0600][Debug  ][XRootD            ] Redirect trace-back:
> [2023-03-10 15:45:37.938750 -0600][Debug  ][XRootD            ] 	0. Redirected from: roots://xrootd-local.unl.edu:1094//store/hello_world.txt to: roots://red-xfer8.unl.edu:1094/
> [2023-03-10 15:45:37.938765 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] URL: roots://red-xfer8.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Host Name: red-xfer8.unl.edu
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 15:45:37.938791 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 15:45:37.938838 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending a read command for handle 0x0 to red-xfer8.unl.edu:1094
> [2023-03-10 15:45:37.938873 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 15:45:37.938892 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ).
> [2023-03-10 15:45:37.938910 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0) through substream 0 expecting answer at 0
> [2023-03-10 15:45:37.938962 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0), 32 bytes
> [2023-03-10 15:45:37.938994 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x205afc0).
> [2023-03-10 15:45:37.939007 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_read (handle: 0x00000000, offset: 0, size: 13) has been successfully sent.
> [2023-03-10 15:45:37.939018 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) from out-queu to in-queue.
> [2023-03-10 15:45:37.939029 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 15:45:37.939104 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4068400] Expecting 13 bytes of message body
> [2023-03-10 15:45:37.939119 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb4068400 size: 8
> [2023-03-10 15:45:37.939129 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4068400] Assigned MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.939139 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
> [2023-03-10 15:45:37.939149 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Will use the raw handler to read body of message 0xb4068400
> [2023-03-10 15:45:37.939181 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb4068400 of 21 bytes
> [2023-03-10 15:45:37.939196 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb4068400.
> [2023-03-10 15:45:37.939256 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 15:45:37.939281 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) with status: [SUCCESS] .
> [2023-03-10 15:45:37.939296 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Parsing the response to kXR_read (handle: 0x00000000, offset: 0, size: 13) as ChunkInfo
> [2023-03-10 15:45:37.939311 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Got state response for message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 15:45:37.939337 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
> hello world!
> [2023-03-10 15:45:37.939371 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Sending a close command for handle 0x0 to red-xfer8.unl.edu:1094
> [2023-03-10 15:45:37.939404 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Sending message kXR_close (handle: 0x00000000)
> [2023-03-10 15:45:37.939421 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] MsgHandler created: 0x205e680 (message: kXR_close (handle: 0x00000000) ).
> [2023-03-10 15:45:37.939439 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Sending message kXR_close (handle: 0x00000000) (0x2058a70) through substream 0 expecting answer at 0
> [2023-03-10 15:45:37.939479 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Wrote a message: kXR_close (handle: 0x00000000) (0x2058a70), 24 bytes
> [2023-03-10 15:45:37.939509 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Successfully sent message: kXR_close (handle: 0x00000000) (0x2058a70).
> [2023-03-10 15:45:37.939523 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Message kXR_close (handle: 0x00000000) has been successfully sent.
> [2023-03-10 15:45:37.939533 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Moving MsgHandler: 0x205e680 (message: kXR_close (handle: 0x00000000) ) from out-queu to in-queue.
> [2023-03-10 15:45:37.939544 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 15:45:37.939700 -0600][Dump   ][XRootDTransport   ] [msg: 0xb4063300] Expecting 0 bytes of message body
> [2023-03-10 15:45:37.939716 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message header for 0xb4063300 size: 8
> [2023-03-10 15:45:37.939726 -0600][Debug  ][ExDbgMsg          ] [msg: 0xb4063300] Assigned MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.939736 -0600][Debug  ][ExDbgMsg          ] [handler: 0x205e680] Removed MsgHandler: 0x205e680 from the in-queue.
> [2023-03-10 15:45:37.939747 -0600][Dump   ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Received message 0xb4063300 of 8 bytes
> [2023-03-10 15:45:37.939758 -0600][Dump   ][PostMaster        ] [red-xfer8.unl.edu:1094] Handling received message: 0xb4063300.
> [2023-03-10 15:45:37.939791 -0600][Dump   ][XRootD            ] [red-xfer8.unl.edu:1094] Got a kXR_ok response to request kXR_close (handle: 0x00000000)
> [2023-03-10 15:45:37.939808 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Calling MsgHandler: 0x205e680 (message: kXR_close (handle: 0x00000000) ) with status: [SUCCESS] .
> [2023-03-10 15:45:37.939824 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Close returned from red-xfer8.unl.edu:1094 with: [SUCCESS]
> [2023-03-10 15:45:37.939835 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=1e31cbb6-5cab-4b1d-aa29-5810685d9316] Items in the fly 0, queued for recovery 0
> [2023-03-10 15:45:37.939855 -0600][Debug  ][ExDbgMsg          ] [red-xfer8.unl.edu:1094] Destroying MsgHandler: 0x205e680.
> [2023-03-10 15:45:37.940452 -0600][Debug  ][JobMgr            ] Stopping the job manager...
> [2023-03-10 15:45:37.940475 -0600][Dump   ][JobMgr            ] Stopping worker #0...
> [2023-03-10 15:45:37.940602 -0600][Dump   ][JobMgr            ] Worker #0 stopped
> [2023-03-10 15:45:37.940620 -0600][Dump   ][JobMgr            ] Stopping worker #1...
> [2023-03-10 15:45:37.940679 -0600][Dump   ][JobMgr            ] Worker #1 stopped
> [2023-03-10 15:45:37.940695 -0600][Dump   ][JobMgr            ] Stopping worker #2...
> [2023-03-10 15:45:37.940752 -0600][Dump   ][JobMgr            ] Worker #2 stopped
> [2023-03-10 15:45:37.940768 -0600][Debug  ][JobMgr            ] Job manager stopped
> [2023-03-10 15:45:37.940779 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
> [2023-03-10 15:45:37.940882 -0600][Debug  ][TaskMgr           ] Task manager stopped
> [2023-03-10 15:45:37.940900 -0600][Debug  ][Poller            ] Stopping the poller...
> [2023-03-10 15:45:37.941133 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://red-xfer8.unl.edu:1094/"
> [2023-03-10 15:45:37.941159 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Closing the socket
> [2023-03-10 15:45:37.941205 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:13868><--><[::ffff:129.93.239.138]:1094> Removing socket from the poller
> [2023-03-10 15:45:37.941281 -0600][Debug  ][PostMaster        ] [red-xfer8.unl.edu:1094] Destroying stream
> [2023-03-10 15:45:37.941300 -0600][Debug  ][AsyncSock         ] [red-xfer8.unl.edu:1094.0] Closing the socket
> [2023-03-10 15:45:37.941319 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 15:45:37.941331 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 15:45:37.941341 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:33348><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 15:45:37.941457 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
> [2023-03-10 15:45:37.941474 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> ```
> </details>
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/xrootd/xrootd/issues/1952#issuecomment-1464519072
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

It indeed did recover. So this issue is hopefully transparent to clients, outside of the delay.

Here's a sample server log for when the issue appeared with `xrdcp roots://...`. It seems to match up with the initial ticket comment when I'd been testing with xrdfs:

```
230310 16:31:05.189706 31158 Xrd_Sched: running main accept inq=0
230310 16:31:05.195279 31158 Xrd_Inet: Accepted connection on port 1094 from 59@t3.unl.edu
230310 16:31:05.203361 31158 Xrd_ProtLoad: matched port 1094 protocol xroot
230310 16:31:05.203416 31158 anon.0:59@t3 Xrd_Poll: FD 59 attached to poller 0; num=134
230310 16:31:05.203432 31158 anon.0:59@t3 Xrootd_Protocol: 0000 req=protocol dlen=0
230310 16:31:05.203440 31158 anon.0:59@t3 Xrootd_Response: 0000 sending 8 data bytes; status=0
230310 16:31:05.203491 31158 anon.0:59@t3 TLS_Accept: Accepting a TLS connection...
230310 16:31:05.228423 31158 XrdLinkXeq: anon.0:59@t3 connection upgraded to TLSv1.2
230310 16:31:05.229509 31158 XrootdXeq: anon.0:59@t3 disc 0:00:00
```

This was with my attempt at setting the server-side logging to maximum. Please let me know if further changes would help:
```
xrootd.trace all
xrd.trace all

xrd.tls /etc/grid-security/xrd/xrdcert.pem /etc/grid-security/xrd/xrdkey.pem
xrd.tlsca noverify
xrootd.tls capable all -data
```

Client command:

```
export XRD_NETWORKSTACK=IPv4
export XRD_LOGLEVEL=Dump
xrdcp roots://xrootd-local.unl.edu//store/hello_world.txt -
```

<details>
  <summary>Client output for the server logs above</summary>

```
[2023-03-10 16:31:05.174083 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.5.3
[2023-03-10 16:31:05.174666 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
[2023-03-10 16:31:05.174826 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
[2023-03-10 16:31:05.174877 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
[2023-03-10 16:31:05.174895 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
[2023-03-10 16:31:05.174906 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
[2023-03-10 16:31:05.174971 -0600][Dump   ][PlugInMgr         ] Processing: /etc/xrootd/client.plugins.d/recorder.conf
[2023-03-10 16:31:05.175033 -0600][Dump   ][PlugInMgr         ] Settings from '/etc/xrootd/client.plugins.d/recorder.conf': url='*', lib='/usr/lib64/libXrdClRecorder-5.so', enable='false'
[2023-03-10 16:31:05.175048 -0600][Debug  ][PlugInMgr         ] Trying to disable plug-in for '*'
[2023-03-10 16:31:05.175133 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
[2023-03-10 16:31:05.175161 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
[2023-03-10 16:31:05.176939 -0600][Dump   ][App               ] Chunk size: 8388608, parallel chunks 4, streams: 1
[2023-03-10 16:31:05.176962 -0600][Dump   ][App               ] Processing source entry: roots://xrootd-local.unl.edu//store/hello_world.txt, type other, target file: -
[2023-03-10 16:31:05.177134 -0600][Dump   ][Utility           ] Adding job with properties: 'checkSumMode' = 'none', 'checkSumPreset' = '', 'checkSumType' = '', 'chunkSize' = '8388608', 'coerce' = '0', 'continue' = '0', 'cpTimeout' = '0', 'delegate' = '0', 'doServer' = '0', 'dynamicSource' = '0', 'force' = '0', 'initTimeout' = '600', 'makeDir' = '0', 'parallelChunks' = '4', 'posc' = '0', 'preserveXAttr' = '0', 'rmOnBadCksum' = '0', 'source' = 'roots://xrootd-local.unl.edu//store/hello_world.txt', 'target' = '-', 'targetIsDir' = '0', 'thirdParty' = 'none', 'tpcTimeout' = '1800', 'xcp' = '0', 'xcpBlockSize' = '134217728', 'xrate' = '0', 'xrateThreshold' = '0', 'zipAppend' = '0', 'zipArchive' = '0'
[2023-03-10 16:31:05.177172 -0600][Debug  ][Utility           ] CopyProcess: 2 jobs to prepare
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] URL: -
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Protocol:  stdio
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Host Name: 
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Port:      0
[2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Path:      -
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] URL: -
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Protocol:  stdio
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Host Name: 
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Port:      0
[2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Path:      -
[2023-03-10 16:31:05.177344 -0600][Debug  ][Utility           ] Creating a classic copy job, from roots://xrootd-local.unl.edu:1094//store/hello_world.txt to stdio://-
[2023-03-10 16:31:05.177375 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
[2023-03-10 16:31:05.177459 -0600][Debug  ][Utility           ] Opening roots://xrootd-local.unl.edu:1094//store/hello_world.txt for reading
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:31:05.177648 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending an open command
[2023-03-10 16:31:05.177695 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
[2023-03-10 16:31:05.177713 -0600][Debug  ][Poller            ] Available pollers: built-in
[2023-03-10 16:31:05.177723 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
[2023-03-10 16:31:05.177733 -0600][Debug  ][Poller            ] Creating poller: built-in
[2023-03-10 16:31:05.177748 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
[2023-03-10 16:31:05.190940 -0600][Debug  ][Poller            ] Using 1 poller threads
[2023-03-10 16:31:05.190963 -0600][Debug  ][TaskMgr           ] Starting the task manager...
[2023-03-10 16:31:05.191032 -0600][Debug  ][TaskMgr           ] Task manager started
[2023-03-10 16:31:05.191058 -0600][Debug  ][JobMgr            ] Starting the job manager...
[2023-03-10 16:31:05.191210 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
[2023-03-10 16:31:05.191233 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-10 16:31:05 -0600]
[2023-03-10 16:31:05.191246 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 16:31:05.191282 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094/
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:31:05.191409 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://xrootd-local.unl.edu:1094/
[2023-03-10 16:31:05.191447 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
[2023-03-10 16:31:05.191474 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" to be run at: [2023-03-10 16:31:20 -0600]
[2023-03-10 16:31:05.191495 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
[2023-03-10 16:31:05.191784 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-10 16:31:05.191829 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-10 16:31:05.191882 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
[2023-03-10 16:31:05.199991 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-10 16:31:05.200049 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 16:31:05.200102 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000950), 44 bytes
[2023-03-10 16:31:05.204472 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
[2023-03-10 16:31:05.204490 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:31:05.204505 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 16:31:05.204521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-10 16:31:05.204542 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
[2023-03-10 16:31:05.204568 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:31:05.204583 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 16:31:05.204600 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-10 16:31:05.205173 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 16:31:05.205206 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:31:05.205968 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:31:05.206001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:20 -0600]
[2023-03-10 16:31:05.221931 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:31:05.226144 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:31:05.226286 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 16:31:05.226354 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 109 bytes
[2023-03-10 16:31:05.226639 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
[2023-03-10 16:31:05.226665 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
[2023-03-10 16:31:05.226679 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-10 16:31:05.226694 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:64262><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-10 16:31:05.226784 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
[2023-03-10 16:31:05.226800 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
[2023-03-10 16:31:05.226813 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-10 16:33:05 -0600]
[2023-03-10 16:31:20.207211 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:31:20.207296 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:35 -0600]
[2023-03-10 16:31:20.207315 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:31:20.207338 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:35 -0600]
[2023-03-10 16:31:35.209873 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:31:35.209953 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:50 -0600]
[2023-03-10 16:31:35.209973 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:31:35.209993 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:50 -0600]
[2023-03-10 16:31:50.212835 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:31:50.212900 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:05 -0600]
[2023-03-10 16:31:50.212936 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:31:50.212958 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:05 -0600]
[2023-03-10 16:32:05.215786 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:32:05.215846 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:20 -0600]
[2023-03-10 16:32:05.215857 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:32:05.215870 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:20 -0600]
[2023-03-10 16:32:20.218888 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:32:20.218963 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:35 -0600]
[2023-03-10 16:32:20.218982 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:32:20.219001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:35 -0600]
[2023-03-10 16:32:35.221853 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:32:35.221925 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:50 -0600]
[2023-03-10 16:32:35.221948 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:32:35.221967 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:50 -0600]
[2023-03-10 16:32:50.224723 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:32:50.224790 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:05 -0600]
[2023-03-10 16:32:50.224809 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:32:50.224828 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:05 -0600]
[2023-03-10 16:33:05.227458 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-10 16:33:05.227909 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
[2023-03-10 16:33:05.227982 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
[2023-03-10 16:33:05.228045 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
[2023-03-10 16:33:05.228082 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
[2023-03-10 16:33:05.228093 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:33:05.228106 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:20 -0600]
[2023-03-10 16:33:05.228114 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
[2023-03-10 16:33:05.228125 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:20 -0600]
[2023-03-10 16:33:05.228361 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
[2023-03-10 16:33:05.228398 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 16:33:05.228448 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 44 bytes
[2023-03-10 16:33:05.228704 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
[2023-03-10 16:33:05.228726 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:05.228744 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 16:33:05.228768 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
[2023-03-10 16:33:05.228790 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
[2023-03-10 16:33:05.228799 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:05.228809 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 16:33:05.228821 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
[2023-03-10 16:33:05.229414 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 16:33:05.229434 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:05.231258 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:05.232929 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:05.233011 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 16:33:05.233056 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40401d0), 109 bytes
[2023-03-10 16:33:05.233451 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 75 bytes of message body
[2023-03-10 16:33:05.233472 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:05.233503 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
[2023-03-10 16:33:05.233521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: 732400007f7900007302000088290000
[2023-03-10 16:33:05.233530 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
[2023-03-10 16:33:05.233540 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
[2023-03-10 16:33:05.234929 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
[2023-03-10 16:33:07.099272 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40584f0), 136 bytes
[2023-03-10 16:33:07.106989 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40439d0] Expecting 3252 bytes of message body
[2023-03-10 16:33:07.107017 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.107081 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
[2023-03-10 16:33:07.107096 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
[2023-03-10 16:33:07.117248 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4057df0), 9932 bytes
[2023-03-10 16:33:07.132028 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40401d0] Expecting 0 bytes of message body
[2023-03-10 16:33:07.132059 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.132076 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
[2023-03-10 16:33:07.132144 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
[2023-03-10 16:33:07.132164 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected (IPv4).
[2023-03-10 16:33:07.132201 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
[2023-03-10 16:33:07.132234 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
[2023-03-10 16:33:07.132249 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
[2023-03-10 16:33:07.132258 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
[2023-03-10 16:33:07.132269 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 16:33:07.132893 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4043630] Expecting 21 bytes of message body
[2023-03-10 16:33:07.132937 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0xf4043630 size: 8
[2023-03-10 16:33:07.132961 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4043630] Assigned MsgHandler: 0x262c680.
[2023-03-10 16:33:07.132977 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
[2023-03-10 16:33:07.133028 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0xf4043630 of 29 bytes
[2023-03-10 16:33:07.133043 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0xf4043630.
[2023-03-10 16:33:07.133218 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got kXR_redirect response to message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ): red-xfer3.unl.edu, port 1094
[2023-03-10 16:33:07.133258 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Current server has been assigned as a load-balancer for message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094/
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] URL: fake://fake:111//fake?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9&
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Protocol:  fake
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Host Name: fake
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Port:      111
[2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Path:      /fake
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] URL: fake://fake:111//store/hello_world.txt
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Protocol:  fake
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Host Name: fake
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Port:      111
[2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:33:07.133479 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Retry at server MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Protocol:  root
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Path:      
[2023-03-10 16:33:07.133545 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://red-xfer3.unl.edu:1094/
[2023-03-10 16:33:07.133569 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
[2023-03-10 16:33:07.133586 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/" to be run at: [2023-03-10 16:33:22 -0600]
[2023-03-10 16:33:07.133598 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
[2023-03-10 16:33:07.133967 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.173]:1094
[2023-03-10 16:33:07.133999 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.173]:1094
[2023-03-10 16:33:07.134039 -0600][Debug  ][Poller            ] Adding socket 0xf0001e00 to the poller
[2023-03-10 16:33:07.134882 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Async connection call returned
[2023-03-10 16:33:07.134937 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
[2023-03-10 16:33:07.134972 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 44 bytes
[2023-03-10 16:33:07.135482 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 8 bytes of message body
[2023-03-10 16:33:07.135518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.135539 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 16 bytes
[2023-03-10 16:33:07.135557 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Got the server hand shake response (type: server [], protocol version 511)
[2023-03-10 16:33:07.135584 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 14 bytes of message body
[2023-03-10 16:33:07.135599 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.135610 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 22 bytes
[2023-03-10 16:33:07.135623 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] kXR_protocol successful (type: server [], protocol version 511)
[2023-03-10 16:33:07.136169 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
[2023-03-10 16:33:07.136187 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:07.138392 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:07.140367 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
[2023-03-10 16:33:07.140474 -0600][Info   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake done.
[2023-03-10 16:33:07.140518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 109 bytes
[2023-03-10 16:33:07.141535 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000be0] Expecting 75 bytes of message body
[2023-03-10 16:33:07.141560 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.142357 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 83 bytes
[2023-03-10 16:33:07.142389 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Logged in, session: a1645e003b0500007c00000084156000
[2023-03-10 16:33:07.142398 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
[2023-03-10 16:33:07.142406 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending authentication data
[2023-03-10 16:33:07.142433 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Trying to authenticate using gsi
[2023-03-10 16:33:07.149042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4076e30), 136 bytes
[2023-03-10 16:33:07.160986 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076610] Expecting 3272 bytes of message body
[2023-03-10 16:33:07.161004 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.161042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 3280 bytes
[2023-03-10 16:33:07.161053 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending more authentication data for gsi
[2023-03-10 16:33:07.171097 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4063b20), 9932 bytes
[2023-03-10 16:33:07.184936 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 0 bytes of message body
[2023-03-10 16:33:07.184964 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
[2023-03-10 16:33:07.184979 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 8 bytes
[2023-03-10 16:33:07.185284 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] XrdSecProtect loaded.
[2023-03-10 16:33:07.185302 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authenticated with gsi.
[2023-03-10 16:33:07.185323 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream 0 connected (IPv4).
[2023-03-10 16:33:07.185353 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
[2023-03-10 16:33:07.185378 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
[2023-03-10 16:33:07.185390 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
[2023-03-10 16:33:07.185398 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
[2023-03-10 16:33:07.185407 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 16:33:07.186155 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4067db0] Expecting 87 bytes of message body
[2023-03-10 16:33:07.186182 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4067db0 size: 8
[2023-03-10 16:33:07.186198 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4067db0] Assigned MsgHandler: 0x262c680.
[2023-03-10 16:33:07.186212 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
[2023-03-10 16:33:07.186640 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4067db0 of 95 bytes
[2023-03-10 16:33:07.186665 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4067db0.
[2023-03-10 16:33:07.186820 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 16:33:07.186848 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) with status: [SUCCESS] .
[2023-03-10 16:33:07.186859 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) as OpenInfo
[2023-03-10 16:33:07.186868 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing StatInfo in response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
[2023-03-10 16:33:07.186909 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Open has returned with status [SUCCESS] 
[2023-03-10 16:33:07.186940 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] successfully opened at red-xfer3.unl.edu:1094, handle: 0x0, session id: 1
[2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] Redirect trace-back:
[2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] 	0. Redirected from: roots://xrootd-local.unl.edu:1094//store/hello_world.txt to: roots://red-xfer3.unl.edu:1094/
[2023-03-10 16:33:07.186988 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Protocol:  roots
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] User Name: 
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Password:  
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Port:      1094
[2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
[2023-03-10 16:33:07.187064 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a read command for handle 0x0 to red-xfer3.unl.edu:1094
[2023-03-10 16:33:07.187093 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 16:33:07.187107 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ).
[2023-03-10 16:33:07.187122 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30) through substream 0 expecting answer at 0
[2023-03-10 16:33:07.187161 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30), 32 bytes
[2023-03-10 16:33:07.187189 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30).
[2023-03-10 16:33:07.187201 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_read (handle: 0x00000000, offset: 0, size: 13) has been successfully sent.
[2023-03-10 16:33:07.187209 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) from out-queu to in-queue.
[2023-03-10 16:33:07.187218 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 16:33:07.187560 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076940] Expecting 13 bytes of message body
[2023-03-10 16:33:07.187583 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4076940 size: 8
[2023-03-10 16:33:07.187596 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4076940] Assigned MsgHandler: 0x262de10.
[2023-03-10 16:33:07.187604 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262de10] Removed MsgHandler: 0x262de10 from the in-queue.
[2023-03-10 16:33:07.187612 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Will use the raw handler to read body of message 0xf4076940
[2023-03-10 16:33:07.187639 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4076940 of 21 bytes
[2023-03-10 16:33:07.187651 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4076940.
[2023-03-10 16:33:07.187827 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 16:33:07.187853 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) with status: [SUCCESS] .
[2023-03-10 16:33:07.187871 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_read (handle: 0x00000000, offset: 0, size: 13) as ChunkInfo
[2023-03-10 16:33:07.187883 -0600][Dump   ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Got state response for message kXR_read (handle: 0x00000000, offset: 0, size: 13)
[2023-03-10 16:33:07.187901 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262de10.
hello world!
[2023-03-10 16:33:07.187952 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a close command for handle 0x0 to red-xfer3.unl.edu:1094
[2023-03-10 16:33:07.187979 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000)
[2023-03-10 16:33:07.187992 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_close (handle: 0x00000000) ).
[2023-03-10 16:33:07.188005 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000) (0x26265d0) through substream 0 expecting answer at 0
[2023-03-10 16:33:07.188043 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_close (handle: 0x00000000) (0x26265d0), 24 bytes
[2023-03-10 16:33:07.188071 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_close (handle: 0x00000000) (0x26265d0).
[2023-03-10 16:33:07.188083 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_close (handle: 0x00000000) has been successfully sent.
[2023-03-10 16:33:07.188091 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) from out-queu to in-queue.
[2023-03-10 16:33:07.188099 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
[2023-03-10 16:33:07.188478 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40767f0] Expecting 0 bytes of message body
[2023-03-10 16:33:07.188501 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf40767f0 size: 8
[2023-03-10 16:33:07.188515 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf40767f0] Assigned MsgHandler: 0x262c680.
[2023-03-10 16:33:07.188528 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
[2023-03-10 16:33:07.188536 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf40767f0 of 8 bytes
[2023-03-10 16:33:07.188545 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf40767f0.
[2023-03-10 16:33:07.188638 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_close (handle: 0x00000000)
[2023-03-10 16:33:07.188663 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) with status: [SUCCESS] .
[2023-03-10 16:33:07.188683 -0600][Debug  ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Close returned from red-xfer3.unl.edu:1094 with: [SUCCESS] 
[2023-03-10 16:33:07.188693 -0600][Dump   ][File              ] [0x2627930@roots://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Items in the fly 0, queued for recovery 0
[2023-03-10 16:33:07.188708 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
[2023-03-10 16:33:07.189233 -0600][Debug  ][JobMgr            ] Stopping the job manager...
[2023-03-10 16:33:07.189252 -0600][Dump   ][JobMgr            ] Stopping worker #0...
[2023-03-10 16:33:07.189416 -0600][Dump   ][JobMgr            ] Worker #0 stopped
[2023-03-10 16:33:07.189429 -0600][Dump   ][JobMgr            ] Stopping worker #1...
[2023-03-10 16:33:07.189468 -0600][Dump   ][JobMgr            ] Worker #1 stopped
[2023-03-10 16:33:07.189478 -0600][Dump   ][JobMgr            ] Stopping worker #2...
[2023-03-10 16:33:07.189511 -0600][Dump   ][JobMgr            ] Worker #2 stopped
[2023-03-10 16:33:07.189521 -0600][Debug  ][JobMgr            ] Job manager stopped
[2023-03-10 16:33:07.189529 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
[2023-03-10 16:33:07.189602 -0600][Debug  ][TaskMgr           ] Task manager stopped
[2023-03-10 16:33:07.189612 -0600][Debug  ][Poller            ] Stopping the poller...
[2023-03-10 16:33:07.189673 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/"
[2023-03-10 16:33:07.189699 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
[2023-03-10 16:33:07.189745 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:8496><--><[::ffff:129.93.239.173]:1094> Removing socket from the poller
[2023-03-10 16:33:07.189824 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Destroying stream
[2023-03-10 16:33:07.189846 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
[2023-03-10 16:33:07.189863 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
[2023-03-10 16:33:07.189873 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
[2023-03-10 16:33:07.189882 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:19674><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
[2023-03-10 16:33:07.189998 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
[2023-03-10 16:33:07.190014 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
```
</details>

It's transparent for xroots but not so for https. There is no recovery for 
that protocol. Your just supposed to hit reload you know.....

On Fri, 10 Mar 2023, jthiltges wrote:

> It indeed did recover. So this issue is hopefully transparent to clients, outside of the delay.
>
> Here's a sample server log for when the issue appeared with `xrdcp roots://...`. It seems to match up with the initial ticket comment when I'd been testing with xrdfs:
>
> ```
> 230310 16:31:05.189706 31158 Xrd_Sched: running main accept inq=0
> 230310 16:31:05.195279 31158 Xrd_Inet: Accepted connection on port 1094 from ***@***.***
> 230310 16:31:05.203361 31158 Xrd_ProtLoad: matched port 1094 protocol xroot
> 230310 16:31:05.203416 31158 ***@***.*** Xrd_Poll: FD 59 attached to poller 0; num=134
> 230310 16:31:05.203432 31158 ***@***.*** Xrootd_Protocol: 0000 req=protocol dlen=0
> 230310 16:31:05.203440 31158 ***@***.*** Xrootd_Response: 0000 sending 8 data bytes; status=0
> 230310 16:31:05.203491 31158 ***@***.*** TLS_Accept: Accepting a TLS connection...
> 230310 16:31:05.228423 31158 XrdLinkXeq: ***@***.*** connection upgraded to TLSv1.2
> 230310 16:31:05.229509 31158 XrootdXeq: ***@***.*** disc 0:00:00
> ```
>
> This was with my attempt at setting the server-side logging to maximum. Please let me know if further changes would help:
> ```
> xrootd.trace all
> xrd.trace all
>
> xrd.tls /etc/grid-security/xrd/xrdcert.pem /etc/grid-security/xrd/xrdkey.pem
> xrd.tlsca noverify
> xrootd.tls capable all -data
> ```
>
> Client command:
>
> ```
> export XRD_NETWORKSTACK=IPv4
> export XRD_LOGLEVEL=Dump
> xrdcp roots://xrootd-local.unl.edu//store/hello_world.txt -
> ```
>
> <details>
>  <summary>Client output for the server logs above</summary>
>
> ```
> [2023-03-10 16:31:05.174083 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.5.3
> [2023-03-10 16:31:05.174666 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
> [2023-03-10 16:31:05.174826 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
> [2023-03-10 16:31:05.174877 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
> [2023-03-10 16:31:05.174895 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
> [2023-03-10 16:31:05.174906 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
> [2023-03-10 16:31:05.174971 -0600][Dump   ][PlugInMgr         ] Processing: /etc/xrootd/client.plugins.d/recorder.conf
> [2023-03-10 16:31:05.175033 -0600][Dump   ][PlugInMgr         ] Settings from '/etc/xrootd/client.plugins.d/recorder.conf': url='*', lib='/usr/lib64/libXrdClRecorder-5.so', enable='false'
> [2023-03-10 16:31:05.175048 -0600][Debug  ][PlugInMgr         ] Trying to disable plug-in for '*'
> [2023-03-10 16:31:05.175133 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
> [2023-03-10 16:31:05.175161 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
> [2023-03-10 16:31:05.176939 -0600][Dump   ][App               ] Chunk size: 8388608, parallel chunks 4, streams: 1
> [2023-03-10 16:31:05.176962 -0600][Dump   ][App               ] Processing source entry: roots://xrootd-local.unl.edu//store/hello_world.txt, type other, target file: -
> [2023-03-10 16:31:05.177134 -0600][Dump   ][Utility           ] Adding job with properties: 'checkSumMode' = 'none', 'checkSumPreset' = '', 'checkSumType' = '', 'chunkSize' = '8388608', 'coerce' = '0', 'continue' = '0', 'cpTimeout' = '0', 'delegate' = '0', 'doServer' = '0', 'dynamicSource' = '0', 'force' = '0', 'initTimeout' = '600', 'makeDir' = '0', 'parallelChunks' = '4', 'posc' = '0', 'preserveXAttr' = '0', 'rmOnBadCksum' = '0', 'source' = 'roots://xrootd-local.unl.edu//store/hello_world.txt', 'target' = '-', 'targetIsDir' = '0', 'thirdParty' = 'none', 'tpcTimeout' = '1800', 'xcp' = '0', 'xcpBlockSize' = '134217728', 'xrate' = '0', 'xrateThreshold' = '0', 'zipAppend' = '0', 'zipArchive' = '0'
> [2023-03-10 16:31:05.177172 -0600][Debug  ][Utility           ] CopyProcess: 2 jobs to prepare
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 16:31:05.177344 -0600][Debug  ][Utility           ] Creating a classic copy job, from roots://xrootd-local.unl.edu:1094//store/hello_world.txt to stdio://-
> [2023-03-10 16:31:05.177375 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
> [2023-03-10 16:31:05.177459 -0600][Debug  ][Utility           ] Opening roots://xrootd-local.unl.edu:1094//store/hello_world.txt for reading
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177648 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending an open command
> [2023-03-10 16:31:05.177695 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
> [2023-03-10 16:31:05.177713 -0600][Debug  ][Poller            ] Available pollers: built-in
> [2023-03-10 16:31:05.177723 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
> [2023-03-10 16:31:05.177733 -0600][Debug  ][Poller            ] Creating poller: built-in
> [2023-03-10 16:31:05.177748 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
> [2023-03-10 16:31:05.190940 -0600][Debug  ][Poller            ] Using 1 poller threads
> [2023-03-10 16:31:05.190963 -0600][Debug  ][TaskMgr           ] Starting the task manager...
> [2023-03-10 16:31:05.191032 -0600][Debug  ][TaskMgr           ] Task manager started
> [2023-03-10 16:31:05.191058 -0600][Debug  ][JobMgr            ] Starting the job manager...
> [2023-03-10 16:31:05.191210 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
> [2023-03-10 16:31:05.191233 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-10 16:31:05 -0600]
> [2023-03-10 16:31:05.191246 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:31:05.191282 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:31:05.191409 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 16:31:05.191447 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 16:31:05.191474 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" to be run at: [2023-03-10 16:31:20 -0600]
> [2023-03-10 16:31:05.191495 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
> [2023-03-10 16:31:05.191784 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 16:31:05.191829 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 16:31:05.191882 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
> [2023-03-10 16:31:05.199991 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:31:05.200049 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:31:05.200102 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000950), 44 bytes
> [2023-03-10 16:31:05.204472 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
> [2023-03-10 16:31:05.204490 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:31:05.204505 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:31:05.204521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 16:31:05.204542 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
> [2023-03-10 16:31:05.204568 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:31:05.204583 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:31:05.204600 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 16:31:05.205173 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:31:05.205206 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.205968 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:05.206001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:20 -0600]
> [2023-03-10 16:31:05.221931 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.226144 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.226286 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:31:05.226354 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 109 bytes
> [2023-03-10 16:31:05.226639 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
> [2023-03-10 16:31:05.226665 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
> [2023-03-10 16:31:05.226679 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:31:05.226694 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:64262><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 16:31:05.226784 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
> [2023-03-10 16:31:05.226800 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
> [2023-03-10 16:31:05.226813 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:31:20.207211 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:20.207296 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:35 -0600]
> [2023-03-10 16:31:20.207315 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:20.207338 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:35 -0600]
> [2023-03-10 16:31:35.209873 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:35.209953 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:50 -0600]
> [2023-03-10 16:31:35.209973 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:35.209993 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:50 -0600]
> [2023-03-10 16:31:50.212835 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:50.212900 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:05 -0600]
> [2023-03-10 16:31:50.212936 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:50.212958 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:05 -0600]
> [2023-03-10 16:32:05.215786 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:05.215846 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:20 -0600]
> [2023-03-10 16:32:05.215857 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:05.215870 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:20 -0600]
> [2023-03-10 16:32:20.218888 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:20.218963 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:35 -0600]
> [2023-03-10 16:32:20.218982 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:20.219001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:35 -0600]
> [2023-03-10 16:32:35.221853 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:35.221925 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:50 -0600]
> [2023-03-10 16:32:35.221948 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:35.221967 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:50 -0600]
> [2023-03-10 16:32:50.224723 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:50.224790 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:32:50.224809 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:50.224828 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:33:05.227458 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 16:33:05.227909 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 16:33:05.227982 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 16:33:05.228045 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
> [2023-03-10 16:33:05.228082 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 16:33:05.228093 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:33:05.228106 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:20 -0600]
> [2023-03-10 16:33:05.228114 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:33:05.228125 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:20 -0600]
> [2023-03-10 16:33:05.228361 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:33:05.228398 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:33:05.228448 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 44 bytes
> [2023-03-10 16:33:05.228704 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
> [2023-03-10 16:33:05.228726 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.228744 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:05.228768 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 16:33:05.228790 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
> [2023-03-10 16:33:05.228799 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.228809 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:05.228821 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 16:33:05.229414 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:33:05.229434 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.231258 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.232929 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.233011 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:33:05.233056 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40401d0), 109 bytes
> [2023-03-10 16:33:05.233451 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 75 bytes of message body
> [2023-03-10 16:33:05.233472 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.233503 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 16:33:05.233521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: 732400007f7900007302000088290000
> [2023-03-10 16:33:05.233530 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 16:33:05.233540 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
> [2023-03-10 16:33:05.234929 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 16:33:07.099272 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40584f0), 136 bytes
> [2023-03-10 16:33:07.106989 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40439d0] Expecting 3252 bytes of message body
> [2023-03-10 16:33:07.107017 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.107081 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
> [2023-03-10 16:33:07.107096 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 16:33:07.117248 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4057df0), 9932 bytes
> [2023-03-10 16:33:07.132028 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40401d0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.132059 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.132076 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 16:33:07.132144 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 16:33:07.132164 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 16:33:07.132201 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
> [2023-03-10 16:33:07.132234 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
> [2023-03-10 16:33:07.132249 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 16:33:07.132258 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.132269 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.132893 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4043630] Expecting 21 bytes of message body
> [2023-03-10 16:33:07.132937 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0xf4043630 size: 8
> [2023-03-10 16:33:07.132961 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4043630] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.132977 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.133028 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0xf4043630 of 29 bytes
> [2023-03-10 16:33:07.133043 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0xf4043630.
> [2023-03-10 16:33:07.133218 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got kXR_redirect response to message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ): red-xfer3.unl.edu, port 1094
> [2023-03-10 16:33:07.133258 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Current server has been assigned as a load-balancer for message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] URL: fake://fake:111//fake?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9&
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Path:      /fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] URL: fake://fake:111//store/hello_world.txt
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133479 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Retry at server MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133545 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133569 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 16:33:07.133586 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/" to be run at: [2023-03-10 16:33:22 -0600]
> [2023-03-10 16:33:07.133598 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.133967 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.173]:1094
> [2023-03-10 16:33:07.133999 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.173]:1094
> [2023-03-10 16:33:07.134039 -0600][Debug  ][Poller            ] Adding socket 0xf0001e00 to the poller
> [2023-03-10 16:33:07.134882 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:33:07.134937 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:33:07.134972 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 44 bytes
> [2023-03-10 16:33:07.135482 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 8 bytes of message body
> [2023-03-10 16:33:07.135518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.135539 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:07.135557 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Got the server hand shake response (type: server [], protocol version 511)
> [2023-03-10 16:33:07.135584 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 14 bytes of message body
> [2023-03-10 16:33:07.135599 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.135610 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 22 bytes
> [2023-03-10 16:33:07.135623 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] kXR_protocol successful (type: server [], protocol version 511)
> [2023-03-10 16:33:07.136169 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:33:07.136187 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.138392 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.140367 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.140474 -0600][Info   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:33:07.140518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 109 bytes
> [2023-03-10 16:33:07.141535 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000be0] Expecting 75 bytes of message body
> [2023-03-10 16:33:07.141560 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.142357 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 16:33:07.142389 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Logged in, session: a1645e003b0500007c00000084156000
> [2023-03-10 16:33:07.142398 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 16:33:07.142406 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending authentication data
> [2023-03-10 16:33:07.142433 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 16:33:07.149042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4076e30), 136 bytes
> [2023-03-10 16:33:07.160986 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076610] Expecting 3272 bytes of message body
> [2023-03-10 16:33:07.161004 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.161042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 3280 bytes
> [2023-03-10 16:33:07.161053 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 16:33:07.171097 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4063b20), 9932 bytes
> [2023-03-10 16:33:07.184936 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.184964 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.184979 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 16:33:07.185284 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] XrdSecProtect loaded.
> [2023-03-10 16:33:07.185302 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 16:33:07.185323 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 16:33:07.185353 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
> [2023-03-10 16:33:07.185378 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
> [2023-03-10 16:33:07.185390 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 16:33:07.185398 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.185407 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.186155 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4067db0] Expecting 87 bytes of message body
> [2023-03-10 16:33:07.186182 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4067db0 size: 8
> [2023-03-10 16:33:07.186198 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4067db0] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.186212 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.186640 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4067db0 of 95 bytes
> [2023-03-10 16:33:07.186665 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4067db0.
> [2023-03-10 16:33:07.186820 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.186848 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.186859 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) as OpenInfo
> [2023-03-10 16:33:07.186868 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing StatInfo in response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.186909 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Open has returned with status [SUCCESS]
> [2023-03-10 16:33:07.186940 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] successfully opened at red-xfer3.unl.edu:1094, handle: 0x0, session id: 1
> [2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] Redirect trace-back:
> [2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] 	0. Redirected from: roots://xrootd-local.unl.edu:1094//store/hello_world.txt to: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.186988 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.187064 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a read command for handle 0x0 to red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.187093 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187107 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ).
> [2023-03-10 16:33:07.187122 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.187161 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30), 32 bytes
> [2023-03-10 16:33:07.187189 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30).
> [2023-03-10 16:33:07.187201 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_read (handle: 0x00000000, offset: 0, size: 13) has been successfully sent.
> [2023-03-10 16:33:07.187209 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.187218 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.187560 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076940] Expecting 13 bytes of message body
> [2023-03-10 16:33:07.187583 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4076940 size: 8
> [2023-03-10 16:33:07.187596 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4076940] Assigned MsgHandler: 0x262de10.
> [2023-03-10 16:33:07.187604 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262de10] Removed MsgHandler: 0x262de10 from the in-queue.
> [2023-03-10 16:33:07.187612 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Will use the raw handler to read body of message 0xf4076940
> [2023-03-10 16:33:07.187639 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4076940 of 21 bytes
> [2023-03-10 16:33:07.187651 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4076940.
> [2023-03-10 16:33:07.187827 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187853 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.187871 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_read (handle: 0x00000000, offset: 0, size: 13) as ChunkInfo
> [2023-03-10 16:33:07.187883 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Got state response for message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187901 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262de10.
> hello world!
> [2023-03-10 16:33:07.187952 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a close command for handle 0x0 to red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.187979 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000)
> [2023-03-10 16:33:07.187992 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_close (handle: 0x00000000) ).
> [2023-03-10 16:33:07.188005 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000) (0x26265d0) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.188043 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_close (handle: 0x00000000) (0x26265d0), 24 bytes
> [2023-03-10 16:33:07.188071 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_close (handle: 0x00000000) (0x26265d0).
> [2023-03-10 16:33:07.188083 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_close (handle: 0x00000000) has been successfully sent.
> [2023-03-10 16:33:07.188091 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.188099 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.188478 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40767f0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.188501 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf40767f0 size: 8
> [2023-03-10 16:33:07.188515 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf40767f0] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.188528 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.188536 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf40767f0 of 8 bytes
> [2023-03-10 16:33:07.188545 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf40767f0.
> [2023-03-10 16:33:07.188638 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_close (handle: 0x00000000)
> [2023-03-10 16:33:07.188663 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.188683 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Close returned from red-xfer3.unl.edu:1094 with: [SUCCESS]
> [2023-03-10 16:33:07.188693 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Items in the fly 0, queued for recovery 0
> [2023-03-10 16:33:07.188708 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.189233 -0600][Debug  ][JobMgr            ] Stopping the job manager...
> [2023-03-10 16:33:07.189252 -0600][Dump   ][JobMgr            ] Stopping worker #0...
> [2023-03-10 16:33:07.189416 -0600][Dump   ][JobMgr            ] Worker #0 stopped
> [2023-03-10 16:33:07.189429 -0600][Dump   ][JobMgr            ] Stopping worker #1...
> [2023-03-10 16:33:07.189468 -0600][Dump   ][JobMgr            ] Worker #1 stopped
> [2023-03-10 16:33:07.189478 -0600][Dump   ][JobMgr            ] Stopping worker #2...
> [2023-03-10 16:33:07.189511 -0600][Dump   ][JobMgr            ] Worker #2 stopped
> [2023-03-10 16:33:07.189521 -0600][Debug  ][JobMgr            ] Job manager stopped
> [2023-03-10 16:33:07.189529 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
> [2023-03-10 16:33:07.189602 -0600][Debug  ][TaskMgr           ] Task manager stopped
> [2023-03-10 16:33:07.189612 -0600][Debug  ][Poller            ] Stopping the poller...
> [2023-03-10 16:33:07.189673 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/"
> [2023-03-10 16:33:07.189699 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189745 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:8496><--><[::ffff:129.93.239.173]:1094> Removing socket from the poller
> [2023-03-10 16:33:07.189824 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Destroying stream
> [2023-03-10 16:33:07.189846 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189863 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:33:07.189873 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189882 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:19674><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 16:33:07.189998 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
> [2023-03-10 16:33:07.190014 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> ```
> </details>
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/xrootd/xrootd/issues/1952#issuecomment-1464603394
> You are receiving this because you commented.
>
> Message ID: ***@***.***>

BTW recovery does not exhonerate xroots or the whole TLS stack. We still 
need the TLS tracing enabled and what the log says when a failure occurs.

On Fri, 10 Mar 2023, jthiltges wrote:

> It indeed did recover. So this issue is hopefully transparent to clients, outside of the delay.
>
> Here's a sample server log for when the issue appeared with `xrdcp roots://...`. It seems to match up with the initial ticket comment when I'd been testing with xrdfs:
>
> ```
> 230310 16:31:05.189706 31158 Xrd_Sched: running main accept inq=0
> 230310 16:31:05.195279 31158 Xrd_Inet: Accepted connection on port 1094 from ***@***.***
> 230310 16:31:05.203361 31158 Xrd_ProtLoad: matched port 1094 protocol xroot
> 230310 16:31:05.203416 31158 ***@***.*** Xrd_Poll: FD 59 attached to poller 0; num=134
> 230310 16:31:05.203432 31158 ***@***.*** Xrootd_Protocol: 0000 req=protocol dlen=0
> 230310 16:31:05.203440 31158 ***@***.*** Xrootd_Response: 0000 sending 8 data bytes; status=0
> 230310 16:31:05.203491 31158 ***@***.*** TLS_Accept: Accepting a TLS connection...
> 230310 16:31:05.228423 31158 XrdLinkXeq: ***@***.*** connection upgraded to TLSv1.2
> 230310 16:31:05.229509 31158 XrootdXeq: ***@***.*** disc 0:00:00
> ```
>
> This was with my attempt at setting the server-side logging to maximum. Please let me know if further changes would help:
> ```
> xrootd.trace all
> xrd.trace all
>
> xrd.tls /etc/grid-security/xrd/xrdcert.pem /etc/grid-security/xrd/xrdkey.pem
> xrd.tlsca noverify
> xrootd.tls capable all -data
> ```
>
> Client command:
>
> ```
> export XRD_NETWORKSTACK=IPv4
> export XRD_LOGLEVEL=Dump
> xrdcp roots://xrootd-local.unl.edu//store/hello_world.txt -
> ```
>
> <details>
>  <summary>Client output for the server logs above</summary>
>
> ```
> [2023-03-10 16:31:05.174083 -0600][Debug  ][Utility           ] Initializing xrootd client version: v5.5.3
> [2023-03-10 16:31:05.174666 -0600][Debug  ][Utility           ] Unable to process user config file: [ERROR] OS Error: no such file or directory
> [2023-03-10 16:31:05.174826 -0600][Info   ][Utility           ] Env: Importing from shell XRD_NETWORKSTACK=IPv4 as networkstack
> [2023-03-10 16:31:05.174877 -0600][Debug  ][PlugInMgr         ] Initializing plug-in manager...
> [2023-03-10 16:31:05.174895 -0600][Debug  ][PlugInMgr         ] No default plug-in, loading plug-in configs...
> [2023-03-10 16:31:05.174906 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /etc/xrootd/client.plugins.d...
> [2023-03-10 16:31:05.174971 -0600][Dump   ][PlugInMgr         ] Processing: /etc/xrootd/client.plugins.d/recorder.conf
> [2023-03-10 16:31:05.175033 -0600][Dump   ][PlugInMgr         ] Settings from '/etc/xrootd/client.plugins.d/recorder.conf': url='*', lib='/usr/lib64/libXrdClRecorder-5.so', enable='false'
> [2023-03-10 16:31:05.175048 -0600][Debug  ][PlugInMgr         ] Trying to disable plug-in for '*'
> [2023-03-10 16:31:05.175133 -0600][Debug  ][PlugInMgr         ] Processing plug-in definitions in /home/swanson/jthiltge/.xrootd/client.plugins.d...
> [2023-03-10 16:31:05.175161 -0600][Debug  ][PlugInMgr         ] Unable to process directory /home/swanson/jthiltge/.xrootd/client.plugins.d: [ERROR] OS Error: no such file or directory
> [2023-03-10 16:31:05.176939 -0600][Dump   ][App               ] Chunk size: 8388608, parallel chunks 4, streams: 1
> [2023-03-10 16:31:05.176962 -0600][Dump   ][App               ] Processing source entry: roots://xrootd-local.unl.edu//store/hello_world.txt, type other, target file: -
> [2023-03-10 16:31:05.177134 -0600][Dump   ][Utility           ] Adding job with properties: 'checkSumMode' = 'none', 'checkSumPreset' = '', 'checkSumType' = '', 'chunkSize' = '8388608', 'coerce' = '0', 'continue' = '0', 'cpTimeout' = '0', 'delegate' = '0', 'doServer' = '0', 'dynamicSource' = '0', 'force' = '0', 'initTimeout' = '600', 'makeDir' = '0', 'parallelChunks' = '4', 'posc' = '0', 'preserveXAttr' = '0', 'rmOnBadCksum' = '0', 'source' = 'roots://xrootd-local.unl.edu//store/hello_world.txt', 'target' = '-', 'targetIsDir' = '0', 'thirdParty' = 'none', 'tpcTimeout' = '1800', 'xcp' = '0', 'xcpBlockSize' = '134217728', 'xrate' = '0', 'xrateThreshold' = '0', 'zipAppend' = '0', 'zipArchive' = '0'
> [2023-03-10 16:31:05.177172 -0600][Debug  ][Utility           ] CopyProcess: 2 jobs to prepare
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177211 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 16:31:05.177244 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu//store/hello_world.txt
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177288 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] URL: -
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Protocol:  stdio
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Host Name:
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Port:      0
> [2023-03-10 16:31:05.177326 -0600][Dump   ][Utility           ] Path:      -
> [2023-03-10 16:31:05.177344 -0600][Debug  ][Utility           ] Creating a classic copy job, from roots://xrootd-local.unl.edu:1094//store/hello_world.txt to stdio://-
> [2023-03-10 16:31:05.177375 -0600][Debug  ][Utility           ] Monitor library name not set. No monitoring
> [2023-03-10 16:31:05.177459 -0600][Debug  ][Utility           ] Opening roots://xrootd-local.unl.edu:1094//store/hello_world.txt for reading
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177496 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177531 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.177563 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:31:05.177648 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending an open command
> [2023-03-10 16:31:05.177695 -0600][Debug  ][Utility           ] Env: trying to get a non-existent string entry: pollerpreference
> [2023-03-10 16:31:05.177713 -0600][Debug  ][Poller            ] Available pollers: built-in
> [2023-03-10 16:31:05.177723 -0600][Debug  ][Poller            ] Attempting to create a poller according to preference: built-in
> [2023-03-10 16:31:05.177733 -0600][Debug  ][Poller            ] Creating poller: built-in
> [2023-03-10 16:31:05.177748 -0600][Debug  ][Poller            ] Creating and starting the built-in poller...
> [2023-03-10 16:31:05.190940 -0600][Debug  ][Poller            ] Using 1 poller threads
> [2023-03-10 16:31:05.190963 -0600][Debug  ][TaskMgr           ] Starting the task manager...
> [2023-03-10 16:31:05.191032 -0600][Debug  ][TaskMgr           ] Task manager started
> [2023-03-10 16:31:05.191058 -0600][Debug  ][JobMgr            ] Starting the job manager...
> [2023-03-10 16:31:05.191210 -0600][Debug  ][JobMgr            ] Job manager started, 3 workers
> [2023-03-10 16:31:05.191233 -0600][Debug  ][TaskMgr           ] Registering task: "FileTimer task" to be run at: [2023-03-10 16:31:05 -0600]
> [2023-03-10 16:31:05.191246 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:31:05.191282 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] URL: xrootd-local.unl.edu:1094
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.191338 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:31:05.191375 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:31:05.191409 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://xrootd-local.unl.edu:1094/
> [2023-03-10 16:31:05.191447 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 16:31:05.191474 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" to be run at: [2023-03-10 16:31:20 -0600]
> [2023-03-10 16:31:05.191495 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
> [2023-03-10 16:31:05.191784 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 16:31:05.191829 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 16:31:05.191882 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
> [2023-03-10 16:31:05.199991 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:31:05.200049 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:31:05.200102 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000950), 44 bytes
> [2023-03-10 16:31:05.204472 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
> [2023-03-10 16:31:05.204490 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:31:05.204505 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:31:05.204521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 16:31:05.204542 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000ab0] Expecting 8 bytes of message body
> [2023-03-10 16:31:05.204568 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:31:05.204583 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:31:05.204600 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 16:31:05.205173 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:31:05.205206 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.205968 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:05.206001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:20 -0600]
> [2023-03-10 16:31:05.221931 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.226144 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:31:05.226286 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:31:05.226354 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 109 bytes
> [2023-03-10 16:31:05.226639 -0600][Error  ][TlsMsg            ] [] TLS error rc=-1 ec=5 (error_syscall) errno=104.
> [2023-03-10 16:31:05.226665 -0600][Error  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Socket error while handshaking: [ERROR] TLS error
> [2023-03-10 16:31:05.226679 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:31:05.226694 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:64262><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 16:31:05.226784 -0600][Error  ][PostMaster        ] [xrootd-local.unl.edu:1094] elapsed = 0, pConnectionWindow = 120 seconds.
> [2023-03-10 16:31:05.226800 -0600][Info   ][PostMaster        ] [xrootd-local.unl.edu:1094] Attempting reconnection in 120 seconds.
> [2023-03-10 16:31:05.226813 -0600][Debug  ][TaskMgr           ] Registering task: "StreamConnectorTask for xrootd-local.unl.edu:1094" to be run at: [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:31:20.207211 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:20.207296 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:35 -0600]
> [2023-03-10 16:31:20.207315 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:20.207338 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:35 -0600]
> [2023-03-10 16:31:35.209873 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:35.209953 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:31:50 -0600]
> [2023-03-10 16:31:35.209973 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:35.209993 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:31:50 -0600]
> [2023-03-10 16:31:50.212835 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:31:50.212900 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:05 -0600]
> [2023-03-10 16:31:50.212936 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:31:50.212958 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:05 -0600]
> [2023-03-10 16:32:05.215786 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:05.215846 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:20 -0600]
> [2023-03-10 16:32:05.215857 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:05.215870 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:20 -0600]
> [2023-03-10 16:32:20.218888 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:20.218963 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:35 -0600]
> [2023-03-10 16:32:20.218982 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:20.219001 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:35 -0600]
> [2023-03-10 16:32:35.221853 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:35.221925 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:32:50 -0600]
> [2023-03-10 16:32:35.221948 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:35.221967 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:32:50 -0600]
> [2023-03-10 16:32:50.224723 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:32:50.224790 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:32:50.224809 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:32:50.224828 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:05 -0600]
> [2023-03-10 16:33:05.227458 -0600][Dump   ][TaskMgr           ] Running task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 16:33:05.227909 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.164]:1094
> [2023-03-10 16:33:05.227982 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.164]:1094
> [2023-03-10 16:33:05.228045 -0600][Debug  ][Poller            ] Adding socket 0x262cbb0 to the poller
> [2023-03-10 16:33:05.228082 -0600][Debug  ][TaskMgr           ] Done with task: "StreamConnectorTask for xrootd-local.unl.edu:1094"
> [2023-03-10 16:33:05.228093 -0600][Dump   ][TaskMgr           ] Running task: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:33:05.228106 -0600][Dump   ][TaskMgr           ] Will rerun task "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/" at [2023-03-10 16:33:20 -0600]
> [2023-03-10 16:33:05.228114 -0600][Dump   ][TaskMgr           ] Running task: "FileTimer task"
> [2023-03-10 16:33:05.228125 -0600][Dump   ][TaskMgr           ] Will rerun task "FileTimer task" at [2023-03-10 16:33:20 -0600]
> [2023-03-10 16:33:05.228361 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:33:05.228398 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:33:05.228448 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 44 bytes
> [2023-03-10 16:33:05.228704 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
> [2023-03-10 16:33:05.228726 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.228744 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:05.228768 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Got the server hand shake response (type: manager [], protocol version 511)
> [2023-03-10 16:33:05.228790 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4020400] Expecting 8 bytes of message body
> [2023-03-10 16:33:05.228799 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.228809 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:05.228821 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] kXR_protocol successful (type: manager [], protocol version 511)
> [2023-03-10 16:33:05.229414 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:33:05.229434 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.231258 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.232929 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:05.233011 -0600][Info   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:33:05.233056 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40401d0), 109 bytes
> [2023-03-10 16:33:05.233451 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 75 bytes of message body
> [2023-03-10 16:33:05.233472 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:05.233503 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 16:33:05.233521 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Logged in, session: 732400007f7900007302000088290000
> [2023-03-10 16:33:05.233530 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 16:33:05.233540 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending authentication data
> [2023-03-10 16:33:05.234929 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 16:33:07.099272 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf40584f0), 136 bytes
> [2023-03-10 16:33:07.106989 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40439d0] Expecting 3252 bytes of message body
> [2023-03-10 16:33:07.107017 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.107081 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 3260 bytes
> [2023-03-10 16:33:07.107096 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 16:33:07.117248 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message:  (0xf4057df0), 9932 bytes
> [2023-03-10 16:33:07.132028 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40401d0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.132059 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.132076 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 16:33:07.132144 -0600][Debug  ][XRootDTransport   ] [xrootd-local.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 16:33:07.132164 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 16:33:07.132201 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
> [2023-03-10 16:33:07.132234 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
> [2023-03-10 16:33:07.132249 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 16:33:07.132258 -0600][Debug  ][ExDbgMsg          ] [xrootd-local.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.132269 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.132893 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4043630] Expecting 21 bytes of message body
> [2023-03-10 16:33:07.132937 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message header for 0xf4043630 size: 8
> [2023-03-10 16:33:07.132961 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4043630] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.132977 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.133028 -0600][Dump   ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Received message 0xf4043630 of 29 bytes
> [2023-03-10 16:33:07.133043 -0600][Dump   ][PostMaster        ] [xrootd-local.unl.edu:1094] Handling received message: 0xf4043630.
> [2023-03-10 16:33:07.133218 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Got kXR_redirect response to message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ): red-xfer3.unl.edu, port 1094
> [2023-03-10 16:33:07.133258 -0600][Dump   ][XRootD            ] [xrootd-local.unl.edu:1094] Current server has been assigned as a load-balancer for message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133301 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] URL: fake://fake:111//fake?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9&
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 16:33:07.133332 -0600][Dump   ][Utility           ] Path:      /fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] URL: fake://fake:111//store/hello_world.txt
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Protocol:  fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Host Name: fake
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Port:      111
> [2023-03-10 16:33:07.133373 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] URL: roots://xrootd-local.unl.edu:1094//store/hello_world.txt
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Host Name: xrootd-local.unl.edu
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133415 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133443 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133479 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Retry at server MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ).
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] URL: red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Protocol:  root
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133502 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.133526 -0600][Dump   ][Utility           ] Path:
> [2023-03-10 16:33:07.133545 -0600][Debug  ][PostMaster        ] Creating new channel to: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.133569 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream parameters: Network Stack: IPv4, Connection Window: 120, ConnectionRetry: 5, Stream Error Window: 1800
> [2023-03-10 16:33:07.133586 -0600][Debug  ][TaskMgr           ] Registering task: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/" to be run at: [2023-03-10 16:33:22 -0600]
> [2023-03-10 16:33:07.133598 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.133967 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Found 1 address(es): [::ffff:129.93.239.173]:1094
> [2023-03-10 16:33:07.133999 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Attempting connection to [::ffff:129.93.239.173]:1094
> [2023-03-10 16:33:07.134039 -0600][Debug  ][Poller            ] Adding socket 0xf0001e00 to the poller
> [2023-03-10 16:33:07.134882 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Async connection call returned
> [2023-03-10 16:33:07.134937 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out the initial hand shake + kXR_protocol
> [2023-03-10 16:33:07.134972 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4000be0), 44 bytes
> [2023-03-10 16:33:07.135482 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 8 bytes of message body
> [2023-03-10 16:33:07.135518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.135539 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 16 bytes
> [2023-03-10 16:33:07.135557 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Got the server hand shake response (type: server [], protocol version 511)
> [2023-03-10 16:33:07.135584 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4039090] Expecting 14 bytes of message body
> [2023-03-10 16:33:07.135599 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.135610 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 22 bytes
> [2023-03-10 16:33:07.135623 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] kXR_protocol successful (type: server [], protocol version 511)
> [2023-03-10 16:33:07.136169 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending out kXR_login request, username: jthiltge, cgi: xrd.cc=us&xrd.tz=-6&xrd.appname=xrdcp&xrd.info=&xrd.hostname=t3.unl.edu&xrd.rn=v5.5.3, dual-stack: true, private IPv4: false, private IPv6: false
> [2023-03-10 16:33:07.136187 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.138392 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.140367 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake exchange.
> [2023-03-10 16:33:07.140474 -0600][Info   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] TLS hand-shake done.
> [2023-03-10 16:33:07.140518 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf40430f0), 109 bytes
> [2023-03-10 16:33:07.141535 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4000be0] Expecting 75 bytes of message body
> [2023-03-10 16:33:07.141560 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.142357 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 83 bytes
> [2023-03-10 16:33:07.142389 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Logged in, session: a1645e003b0500007c00000084156000
> [2023-03-10 16:33:07.142398 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authentication is required: &P=gsi,v:10600,c:ssl,ca:ba240aa8.0|f5f0dfc2.0&P=ztn,0:4096:
> [2023-03-10 16:33:07.142406 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending authentication data
> [2023-03-10 16:33:07.142433 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Trying to authenticate using gsi
> [2023-03-10 16:33:07.149042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4076e30), 136 bytes
> [2023-03-10 16:33:07.160986 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076610] Expecting 3272 bytes of message body
> [2023-03-10 16:33:07.161004 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.161042 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 3280 bytes
> [2023-03-10 16:33:07.161053 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Sending more authentication data for gsi
> [2023-03-10 16:33:07.171097 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message:  (0xf4063b20), 9932 bytes
> [2023-03-10 16:33:07.184936 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40430f0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.184964 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header, size: 8
> [2023-03-10 16:33:07.184979 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received a message of 8 bytes
> [2023-03-10 16:33:07.185284 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] XrdSecProtect loaded.
> [2023-03-10 16:33:07.185302 -0600][Debug  ][XRootDTransport   ] [red-xfer3.unl.edu:1094.0] Authenticated with gsi.
> [2023-03-10 16:33:07.185323 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Stream 0 connected (IPv4).
> [2023-03-10 16:33:07.185353 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0), 46 bytes
> [2023-03-10 16:33:07.185378 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) (0x2626bb0).
> [2023-03-10 16:33:07.185390 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) has been successfully sent.
> [2023-03-10 16:33:07.185398 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.185407 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.186155 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4067db0] Expecting 87 bytes of message body
> [2023-03-10 16:33:07.186182 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4067db0 size: 8
> [2023-03-10 16:33:07.186198 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4067db0] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.186212 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.186640 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4067db0 of 95 bytes
> [2023-03-10 16:33:07.186665 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4067db0.
> [2023-03-10 16:33:07.186820 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.186848 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.186859 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat ) as OpenInfo
> [2023-03-10 16:33:07.186868 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing StatInfo in response to kXR_open (file: /store/hello_world.txt, mode: 00, flags: kXR_open_read kXR_async kXR_retstat )
> [2023-03-10 16:33:07.186909 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Open has returned with status [SUCCESS]
> [2023-03-10 16:33:07.186940 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] successfully opened at red-xfer3.unl.edu:1094, handle: 0x0, session id: 1
> [2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] Redirect trace-back:
> [2023-03-10 16:33:07.186976 -0600][Debug  ][XRootD            ] 	0. Redirected from: roots://xrootd-local.unl.edu:1094//store/hello_world.txt to: roots://red-xfer3.unl.edu:1094/
> [2023-03-10 16:33:07.186988 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] URL: roots://red-xfer3.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Protocol:  roots
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] User Name:
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Password:
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Host Name: red-xfer3.unl.edu
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Port:      1094
> [2023-03-10 16:33:07.187022 -0600][Dump   ][Utility           ] Path:      /store/hello_world.txt
> [2023-03-10 16:33:07.187064 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a read command for handle 0x0 to red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.187093 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187107 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ).
> [2023-03-10 16:33:07.187122 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.187161 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30), 32 bytes
> [2023-03-10 16:33:07.187189 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_read (handle: 0x00000000, offset: 0, size: 13) (0x262da30).
> [2023-03-10 16:33:07.187201 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_read (handle: 0x00000000, offset: 0, size: 13) has been successfully sent.
> [2023-03-10 16:33:07.187209 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.187218 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.187560 -0600][Dump   ][XRootDTransport   ] [msg: 0xf4076940] Expecting 13 bytes of message body
> [2023-03-10 16:33:07.187583 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf4076940 size: 8
> [2023-03-10 16:33:07.187596 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf4076940] Assigned MsgHandler: 0x262de10.
> [2023-03-10 16:33:07.187604 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262de10] Removed MsgHandler: 0x262de10 from the in-queue.
> [2023-03-10 16:33:07.187612 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Will use the raw handler to read body of message 0xf4076940
> [2023-03-10 16:33:07.187639 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf4076940 of 21 bytes
> [2023-03-10 16:33:07.187651 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf4076940.
> [2023-03-10 16:33:07.187827 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187853 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262de10 (message: kXR_read (handle: 0x00000000, offset: 0, size: 13) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.187871 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Parsing the response to kXR_read (handle: 0x00000000, offset: 0, size: 13) as ChunkInfo
> [2023-03-10 16:33:07.187883 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Got state response for message kXR_read (handle: 0x00000000, offset: 0, size: 13)
> [2023-03-10 16:33:07.187901 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262de10.
> hello world!
> [2023-03-10 16:33:07.187952 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Sending a close command for handle 0x0 to red-xfer3.unl.edu:1094
> [2023-03-10 16:33:07.187979 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000)
> [2023-03-10 16:33:07.187992 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] MsgHandler created: 0x262c680 (message: kXR_close (handle: 0x00000000) ).
> [2023-03-10 16:33:07.188005 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Sending message kXR_close (handle: 0x00000000) (0x26265d0) through substream 0 expecting answer at 0
> [2023-03-10 16:33:07.188043 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Wrote a message: kXR_close (handle: 0x00000000) (0x26265d0), 24 bytes
> [2023-03-10 16:33:07.188071 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Successfully sent message: kXR_close (handle: 0x00000000) (0x26265d0).
> [2023-03-10 16:33:07.188083 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Message kXR_close (handle: 0x00000000) has been successfully sent.
> [2023-03-10 16:33:07.188091 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Moving MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) from out-queu to in-queue.
> [2023-03-10 16:33:07.188099 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094.0] All messages consumed, disable uplink
> [2023-03-10 16:33:07.188478 -0600][Dump   ][XRootDTransport   ] [msg: 0xf40767f0] Expecting 0 bytes of message body
> [2023-03-10 16:33:07.188501 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message header for 0xf40767f0 size: 8
> [2023-03-10 16:33:07.188515 -0600][Debug  ][ExDbgMsg          ] [msg: 0xf40767f0] Assigned MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.188528 -0600][Debug  ][ExDbgMsg          ] [handler: 0x262c680] Removed MsgHandler: 0x262c680 from the in-queue.
> [2023-03-10 16:33:07.188536 -0600][Dump   ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Received message 0xf40767f0 of 8 bytes
> [2023-03-10 16:33:07.188545 -0600][Dump   ][PostMaster        ] [red-xfer3.unl.edu:1094] Handling received message: 0xf40767f0.
> [2023-03-10 16:33:07.188638 -0600][Dump   ][XRootD            ] [red-xfer3.unl.edu:1094] Got a kXR_ok response to request kXR_close (handle: 0x00000000)
> [2023-03-10 16:33:07.188663 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Calling MsgHandler: 0x262c680 (message: kXR_close (handle: 0x00000000) ) with status: [SUCCESS] .
> [2023-03-10 16:33:07.188683 -0600][Debug  ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Close returned from red-xfer3.unl.edu:1094 with: [SUCCESS]
> [2023-03-10 16:33:07.188693 -0600][Dump   ][File              ] ***@***.***://xrootd-local.unl.edu:1094//store/hello_world.txt?xrdcl.requuid=49bd656c-2498-4298-b14d-b3036dc47ad9] Items in the fly 0, queued for recovery 0
> [2023-03-10 16:33:07.188708 -0600][Debug  ][ExDbgMsg          ] [red-xfer3.unl.edu:1094] Destroying MsgHandler: 0x262c680.
> [2023-03-10 16:33:07.189233 -0600][Debug  ][JobMgr            ] Stopping the job manager...
> [2023-03-10 16:33:07.189252 -0600][Dump   ][JobMgr            ] Stopping worker #0...
> [2023-03-10 16:33:07.189416 -0600][Dump   ][JobMgr            ] Worker #0 stopped
> [2023-03-10 16:33:07.189429 -0600][Dump   ][JobMgr            ] Stopping worker #1...
> [2023-03-10 16:33:07.189468 -0600][Dump   ][JobMgr            ] Worker #1 stopped
> [2023-03-10 16:33:07.189478 -0600][Dump   ][JobMgr            ] Stopping worker #2...
> [2023-03-10 16:33:07.189511 -0600][Dump   ][JobMgr            ] Worker #2 stopped
> [2023-03-10 16:33:07.189521 -0600][Debug  ][JobMgr            ] Job manager stopped
> [2023-03-10 16:33:07.189529 -0600][Debug  ][TaskMgr           ] Stopping the task manager...
> [2023-03-10 16:33:07.189602 -0600][Debug  ][TaskMgr           ] Task manager stopped
> [2023-03-10 16:33:07.189612 -0600][Debug  ][Poller            ] Stopping the poller...
> [2023-03-10 16:33:07.189673 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://red-xfer3.unl.edu:1094/"
> [2023-03-10 16:33:07.189699 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189745 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:8496><--><[::ffff:129.93.239.173]:1094> Removing socket from the poller
> [2023-03-10 16:33:07.189824 -0600][Debug  ][PostMaster        ] [red-xfer3.unl.edu:1094] Destroying stream
> [2023-03-10 16:33:07.189846 -0600][Debug  ][AsyncSock         ] [red-xfer3.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189863 -0600][Debug  ][TaskMgr           ] Requesting unregistration of: "TickGeneratorTask for: roots://xrootd-local.unl.edu:1094/"
> [2023-03-10 16:33:07.189873 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> [2023-03-10 16:33:07.189882 -0600][Debug  ][Poller            ] <[::ffff:129.93.239.166]:19674><--><[::ffff:129.93.239.164]:1094> Removing socket from the poller
> [2023-03-10 16:33:07.189998 -0600][Debug  ][PostMaster        ] [xrootd-local.unl.edu:1094] Destroying stream
> [2023-03-10 16:33:07.190014 -0600][Debug  ][AsyncSock         ] [xrootd-local.unl.edu:1094.0] Closing the socket
> ```
> </details>
>
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/xrootd/xrootd/issues/1952#issuecomment-1464603394
> You are receiving this because you commented.
>
> Message ID: ***@***.***>

--- a/src/XrdTls/XrdTlsSocket.cc
+++ b/src/XrdTls/XrdTlsSocket.cc
@@ -505,6 +505,11 @@ const char *XrdTlsSocket::Init( XrdTlsContext &ctx, int sfd,
    if (hsm) pImpl->hsNoBlock = false;
       else  pImpl->hsNoBlock = true;
 
+// Reset the handshake and fatal error indicators
+//
+   pImpl->hsDone = false;
+   pImpl->fatal = 0;
+
 // The glories of OpenSSL require that we do some fancy footwork with the
 // handshake timeout. If there is one and this is a server and the server
 // wants blocking reads, we initially set the socket as non-blocking as the
@@ -536,6 +541,7 @@ const char *XrdTlsSocket::Init( XrdTlsContext &ctx, int sfd,
   
 XrdTls::RC XrdTlsSocket::Peek( char *buffer, size_t size, int &bytesPeek )
   {
+   EPNAME("Peek");
    XrdSysMutexHelper mHelper;
    int ssler;
 
@@ -550,7 +556,10 @@ XrdTls::RC XrdTlsSocket::Peek( char *buffer, size_t size, int &bytesPeek )
     // SEGV when called after such an error.
     //------------------------------------------------------------------------
 
-    if (pImpl->fatal) return (XrdTls::RC)pImpl->fatal;
+    if (pImpl->fatal)
+       {DBG_SIO("Failing due to previous error, fatal=" << (int)pImpl->fatal);
+        return (XrdTls::RC)pImpl->fatal;
+       }
 
     //------------------------------------------------------------------------
     // If necessary, SSL_read() will negotiate a TLS/SSL session, so we don't
@@ -640,7 +649,10 @@ XrdTls::RC XrdTlsSocket::Read( char *buffer, size_t size, int &bytesRead )
     // SEGV when called after such an error.
     //------------------------------------------------------------------------
 
-    if (pImpl->fatal) return (XrdTls::RC)pImpl->fatal;
+    if (pImpl->fatal) 
+       {DBG_SIO("Failing due to previous error, fatal=" << (int)pImpl->fatal);
+        return (XrdTls::RC)pImpl->fatal;
+       }
 
     //------------------------------------------------------------------------
     // If necessary, SSL_read() will negotiate a TLS/SSL session, so we don't
@@ -785,7 +797,10 @@ XrdTls::RC XrdTlsSocket::Write( const char *buffer, size_t size,
     // SEGV when called after such an error.
     //------------------------------------------------------------------------
 
-    if (pImpl->fatal) return (XrdTls::RC)pImpl->fatal;
+    if (pImpl->fatal)
+       {DBG_SIO("Failing due to previous error, fatal=" << (int)pImpl->fatal);
+        return (XrdTls::RC)pImpl->fatal;
+       }
 
     //------------------------------------------------------------------------
     // If necessary, SSL_write() will negotiate a TLS/SSL session, so we don't

